// @bookingcom/bui-assets-react/genius/GeniusPlusLogoWrapped.js
import * as React from "react";
function GeniusPlusLogoWrapped() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    viewBox: "0 0 217.25 161.52"
  }, /*#__PURE__*/React.createElement("defs", null, /*#__PURE__*/React.createElement("style", null, ".cls-1 {\n        fill: url(#linear-gradient);\n      }\n\n      .cls-1, .cls-2 {\n        fill-rule: evenodd;\n      }\n\n      .cls-3, .cls-2 {\n        fill: #003b95;\n      }\n\n      .cls-4 {\n        fill: #fdba02;\n      }\n\n      .cls-5 {\n        fill: #febb02;\n      }"), /*#__PURE__*/React.createElement("linearGradient", {
    id: "linear-gradient",
    x1: 108.7,
    x2: 110.79,
    y1: -26.91,
    y2: 187.41,
    gradientUnits: "userSpaceOnUse"
  }, /*#__PURE__*/React.createElement("stop", {
    offset: 0.44,
    stopColor: "#ffd66c"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.45,
    stopColor: "#fed468"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.51,
    stopColor: "#fecd48"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.58,
    stopColor: "#fec62e"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.65,
    stopColor: "#fec11a"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.73,
    stopColor: "#febd0c"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 0.83,
    stopColor: "#febb04"
  }), /*#__PURE__*/React.createElement("stop", {
    offset: 1,
    stopColor: "#febb02"
  }))), /*#__PURE__*/React.createElement("g", {
    id: "Layer_1",
    "data-name": "Layer 1"
  }, /*#__PURE__*/React.createElement("path", {
    d: "M129.55,7.14c0,1.95,.69,3.63,2.08,5.03,1.39,1.4,3.05,2.11,4.98,2.11s3.59-.71,4.98-2.11c1.39-1.41,2.08-3.08,2.08-5.03s-.7-3.63-2.08-5.03c-1.39-1.41-3.05-2.11-4.98-2.11s-3.59,.7-4.98,2.11c-1.39,1.41-2.08,3.08-2.08,5.03Z",
    className: "cls-4"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M48.8,49.55c.55-.46,.86-1.13,.86-1.84V26.1c0-.65-.53-1.19-1.18-1.19H31.44c-.31,0-.61,.13-.84,.35-.21,.23-.34,.52-.34,.84v7.47c0,.32,.13,.62,.34,.84,.22,.23,.53,.35,.84,.35h8.65v7.71c-2.98,1.99-6.61,3.15-10.53,3.15-10.09,0-18.28-7.73-18.28-17.25S19.46,11.12,29.55,11.12c4.72,0,9.02,1.69,12.27,4.47,.24,.2,.54,.31,.86,.29,.32-.02,.61-.18,.81-.43,1.15-1.4,3.31-4.03,4.74-5.77,.25-.31,.48-.58,.68-.82,.2-.25,.29-.56,.26-.88-.02-.31-.18-.59-.42-.8C43.59,3.01,36.88,.49,29.55,.49,13.24,.49,0,12.98,0,28.38s13.24,27.88,29.55,27.88c7.34,0,14.08-2.52,19.25-6.7Z",
    className: "cls-3"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M88.58,39.75c.79,0,1.18-.4,1.18-1.2v-2.4c0-5.25-1.58-9.62-4.73-13.1-3.15-3.48-7.52-5.22-13.09-5.22-5.03,0-9.39,1.8-13.09,5.4-3.7,3.6-5.55,8.21-5.55,13.81s1.86,10.21,5.58,13.81c3.73,3.6,8.25,5.4,13.57,5.4,7.05,0,12.37-3.08,15.96-9.23,.4-.65,.22-1.1-.51-1.35l-6.36-2.7c-.64-.3-1.11-.23-1.41,.23-1.78,2.8-4.33,4.2-7.68,4.2-2.46,0-4.5-.7-6.1-2.1-1.6-1.4-2.67-3.25-3.22-5.56h25.44Zm-16.64-13.44c2.12,0,3.96,.59,5.55,1.76,1.58,1.17,2.53,2.61,2.88,4.32h-17c.64-1.8,1.69-3.27,3.14-4.39,1.45-1.13,3.27-1.68,5.44-1.68Z",
    className: "cls-2"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M123.66,21.25c2.24,2.28,3.36,5.49,3.36,9.65v23.27c0,.8-.4,1.2-1.19,1.2h-7.46c-.79,0-1.19-.4-1.19-1.2v-20.64c0-4.51-2.04-6.76-6.14-6.76-1.38,0-2.76,.43-4.14,1.28-1.38,.85-2.44,1.77-3.18,2.77v23.35c0,.8-.42,1.2-1.26,1.2h-7.39c-.83,0-1.26-.4-1.26-1.2V20.01c0-.8,.42-1.2,1.26-1.2h6.07c.84,0,1.38,.38,1.62,1.13l.96,2.1c3.15-2.8,6.6-4.2,10.35-4.2,4.14,0,7.33,1.14,9.58,3.42Z",
    className: "cls-3"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M131.69,54.16c0,.8,.41,1.2,1.26,1.2h7.39c.84,0,1.26-.4,1.26-1.2V20.01c0-.8-.41-1.2-1.26-1.2h-7.39c-.84,0-1.26,.4-1.26,1.2V54.16Z",
    className: "cls-3"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M149.88,52.4c-2.41-2.57-3.62-6.12-3.62-10.62V20.01c0-.8,.42-1.2,1.26-1.2h7.39c.84,0,1.26,.4,1.26,1.2v20.57c0,2.05,.53,3.71,1.59,4.96,1.06,1.25,2.55,1.87,4.47,1.87,3.05,0,5.52-1.38,7.39-4.13V20.01c0-.8,.39-1.2,1.18-1.2h7.47c.79,0,1.19,.4,1.19,1.2V54.16c0,.8-.4,1.2-1.19,1.2h-6.06c-.84,0-1.41-.35-1.7-1.05l-.89-2.18c-.99,.9-1.84,1.6-2.55,2.1-.71,.5-1.72,.97-3.03,1.39-1.31,.42-2.75,.63-4.33,.63-4.14,0-7.42-1.29-9.83-3.87Z",
    className: "cls-3"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M205.85,44.7c0,1.1-.53,1.98-1.59,2.63-1.06,.65-2.45,.97-4.17,.97-2.07,0-3.85-.5-5.33-1.5-1.48-1-2.51-2.28-3.11-3.83-.29-.75-.81-.95-1.55-.6l-5.62,2.55c-.79,.36-1.01,.9-.67,1.65,1.13,2.8,3.09,5.12,5.88,6.94,2.79,1.83,6.32,2.74,10.61,2.74s7.64-1.07,10.5-3.23c2.86-2.15,4.29-5.08,4.29-8.78,0-6.11-4.24-9.89-12.72-11.34-3.11-.55-5.33-1.17-6.69-1.84-1.35-.68-2.03-1.54-2.03-2.59,0-.9,.53-1.62,1.59-2.14,1.06-.53,2.48-.79,4.25-.79,2.86,0,5.35,1.13,7.47,3.38,.59,.6,1.19,.68,1.78,.23l3.99-3.75c.69-.55,.77-1.16,.22-1.8-3.11-3.85-7.67-5.78-13.68-5.78-4.29,0-7.71,.99-10.27,2.96-2.56,1.97-3.85,4.64-3.85,7.99,0,2.8,1.02,5.15,3.07,7.02,2.04,1.88,4.82,3.14,8.31,3.79,3.45,.65,5.86,1.35,7.25,2.1,1.38,.75,2.07,1.75,2.07,3Z",
    className: "cls-3"
  })), /*#__PURE__*/React.createElement("g", {
    id: "Layer_3",
    "data-name": "Layer 3"
  }, /*#__PURE__*/React.createElement("path", {
    d: "M25.68 88.14H194.01000000000002V141.37H25.68z",
    className: "cls-3"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M51.99,102.58c-.11,0-.2,.09-.2,.2v10.31c0,.11,.09,.2,.2,.2h7.08c1.72,0,3.18-.46,4.24-1.52,1.06-.99,1.65-2.25,1.65-3.83s-.6-2.78-1.65-3.83c-1.06-.99-2.52-1.52-4.24-1.52h-7.08Z",
    className: "cls-5"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M49.43,68.62c-25.69,0-46.51,20.8-46.51,46.45s20.82,46.45,46.51,46.45h121.31c25.69,0,46.51-20.8,46.51-46.45s-20.82-46.45-46.51-46.45H49.43Zm1.3,67.74h-7.35c-.73,0-1.06-.33-1.06-1.06v-13.21h-13.4c-.49,0-.89-.4-.89-.89v-7.02c0-.49,.4-.89,.89-.89h13.4v-18.38c0-.73,.33-1.12,1.06-1.12h15.56c4.5,0,8.21,1.26,11.12,3.7,2.91,2.51,4.37,5.95,4.37,10.45s-1.46,8-4.37,10.45c-2.91,2.51-6.62,3.7-11.12,3.7h-7.15v13.21c0,.73-.4,1.06-1.06,1.06Zm113.01-18.5c5.58,1.44,8.4,3.45,8.4,6.09,0,1.38-.63,2.47-1.78,3.39-1.15,.92-2.82,1.38-4.89,1.38-4.14,0-7.07-2.01-8.74-6.15-.23-.63-.69-.8-1.27-.57l-5.41,2.47c-.58,.29-.81,.69-.63,1.26,1.04,3.22,2.93,5.74,5.69,7.7,2.76,1.95,6.15,2.93,10.18,2.93,4.26,0,7.88-1.21,10.76-3.68,2.88-2.47,4.37-5.63,4.37-9.48,0-3.16-1.09-5.69-3.16-7.64-2.07-1.9-5.12-3.33-9.15-4.31-3.28-.75-5.64-1.61-7.07-2.53-1.5-.92-2.19-2.07-2.19-3.56,0-1.15,.58-2.07,1.78-2.87,1.21-.8,2.7-1.21,4.49-1.21,3.34,0,5.92,1.61,7.88,4.77,.23,.46,.63,.52,1.15,.17l4.66-3.39c.52-.34,.63-.75,.4-1.21-1.55-2.53-3.39-4.42-5.58-5.69-2.19-1.26-5.06-1.95-8.51-1.95-4.03,0-7.42,1.09-10.12,3.27-2.76,2.18-4.08,4.94-4.08,8.33s1.09,6.03,3.34,7.98c2.19,1.95,5.35,3.45,9.49,4.48Zm-42.78-22.4v23.49s0,8.01,6.97,8.01,7.09-8.01,7.09-8.01v-23.49c0-.63,.29-.98,.92-.98h7.58c.63,0,.98,.34,.98,.98v23.49c0,5.46-1.55,9.71-4.6,12.81-3.11,3.1-7.07,4.6-11.91,4.6s-8.86-1.49-11.91-4.6c-3.11-3.1-4.6-7.35-4.6-12.81v-23.49c0-.63,.29-.98,.98-.98h7.52c.63,0,.98,.34,.98,.98Zm-40.74,40.21h25.42c.63,0,.98-.29,.98-.92v-6.91c0-.57-.35-.92-.98-.92h-16.87v-31.46c0-.63-.35-.98-.92-.98h-7.63c-.63,0-.92,.34-.92,.98v39.29c0,.63,.29,.92,.92,.92Z",
    className: "cls-1"
  })));
}
export default GeniusPlusLogoWrapped;
