// @bookingcom/bui-assets-react/genius/GeniusVipBadge.js
import * as React from "react";
function GeniusVipBadge() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    id: "Layer_1",
    x: 0,
    y: 0,
    enableBackground: "new 0 0 80 80",
    viewBox: "0 0 80 80"
  }, /*#__PURE__*/React.createElement("style", null, ".st0{fill:#FEBB02;}\n\t.st1{fill:#003B95;}"), /*#__PURE__*/React.createElement("path", {
    d: "M8,12h64c4.4,0,8,3.6,8,8v40c0,4.4-3.6,8-8,8H8c-4.4,0-8-3.6-8-8V20C0,15.6,3.6,12,8,12z",
    className: "st0"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M25.2,54h-6.8L8,26h7.6L22,46l6.4-20H36L25.2,54z",
    className: "st1"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M38.4,26h6.8v28h-6.8V26z",
    className: "st1"
  }), /*#__PURE__*/React.createElement("path", {
    d: "M72.3,34.7c0,1.7-0.3,3-0.9,4.1c-0.6,1.1-1.4,2-2.4,2.7s-2.1,1.2-3.5,1.5c-1.3,0.3-2.7,0.5-4.1,0.5h-3.4V54 h-6.7V26h10.2c1.5,0,2.9,0.2,4.2,0.5c1.3,0.3,2.4,0.7,3.5,1.5c1,0.7,1.8,1.6,2.3,2.6C72,31.7,72.3,33.1,72.3,34.7L72.3,34.7z M65.6,34.7c0-0.7-0.1-1.2-0.4-1.6c-0.3-0.5-0.7-0.7-1-1c-0.5-0.3-0.9-0.4-1.5-0.5c-0.6-0.1-1.1-0.1-1.7-0.1h-2.9v6.6h2.8 c0.7,0,1.2-0.1,1.8-0.2c0.6-0.1,1.1-0.3,1.5-0.6c0.5-0.3,0.8-0.6,1.1-1C65.5,35.9,65.6,35.3,65.6,34.7L65.6,34.7z",
    className: "st1"
  }));
}
export default GeniusVipBadge;
