// @bookingcom/bui-assets-react/build.js
"use strict";
var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
    return new (P || (P = Promise))(function (resolve, reject) {
        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
        step((generator = generator.apply(thisArg, _arguments || [])).next());
    });
};
var __generator = (this && this.__generator) || function (thisArg, body) {
    var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g;
    return g = { next: verb(0), "throw": verb(1), "return": verb(2) }, typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
    function verb(n) { return function (v) { return step([n, v]); }; }
    function step(op) {
        if (f) throw new TypeError("Generator is already executing.");
        while (g && (g = 0, op[0] && (_ = 0)), _) try {
            if (f = 1, y && (t = op[0] & 2 ? y["return"] : op[0] ? y["throw"] || ((t = y["return"]) && t.call(y), 0) : y.next) && !(t = t.call(y, op[1])).done) return t;
            if (y = 0, t) op = [op[0] & 2, t.value];
            switch (op[0]) {
                case 0: case 1: t = op; break;
                case 4: _.label++; return { value: op[1], done: false };
                case 5: _.label++; y = op[1]; op = [0]; continue;
                case 7: op = _.ops.pop(); _.trys.pop(); continue;
                default:
                    if (!(t = _.trys, t = t.length > 0 && t[t.length - 1]) && (op[0] === 6 || op[0] === 2)) { _ = 0; continue; }
                    if (op[0] === 3 && (!t || (op[1] > t[0] && op[1] < t[3]))) { _.label = op[1]; break; }
                    if (op[0] === 6 && _.label < t[1]) { _.label = t[1]; t = op; break; }
                    if (t && _.label < t[2]) { _.label = t[2]; _.ops.push(op); break; }
                    if (t[2]) _.ops.pop();
                    _.trys.pop(); continue;
            }
            op = body.call(thisArg, _);
        } catch (e) { op = [6, e]; y = 0; } finally { f = t = 0; }
        if (op[0] & 5) throw op[1]; return { value: op[0] ? op[1] : void 0, done: true };
    }
};
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
var path_1 = __importDefault(require("path"));
var core_1 = require("@babel/core");
var core_2 = __importDefault(require("@svgr/core"));
var bui_assets_fs_1 = require("@bookingcom/bui-assets-fs");
var getTypeDeclaration = function (componentName) { return "\nimport * as React from \"react\";\ndeclare const ".concat(componentName, ": React.FC;\nexport default ").concat(componentName, ";\n"); };
var transformToComponent = function (componentName, svg) {
    var jsxComponent = core_2.default.sync(svg, { prettier: true, svgo: false, expandProps: false }, { componentName: componentName });
    var jsComponent = (0, core_1.transformSync)(jsxComponent, {
        plugins: ["@babel/plugin-transform-react-jsx"],
    });
    return jsComponent === null || jsComponent === void 0 ? void 0 : jsComponent.code;
};
var assetNames = [];
(0, bui_assets_fs_1.walkAssets)({
    onIconComponent: function (_a) {
        var assetName = _a.assetName, assetCode = _a.assetCode, packageSetName = _a.packageSetName, setType = _a.setType;
        return __awaiter(void 0, void 0, void 0, function () {
            var componentName, componentCode, typeDeclarationCode;
            return __generator(this, function (_b) {
                componentName = (0, bui_assets_fs_1.prepareAssetName)(assetName, setType);
                componentCode = transformToComponent(componentName, assetCode);
                typeDeclarationCode = getTypeDeclaration(componentName);
                if (!componentCode) {
                    throw new Error("BUI Assets React: Component \"".concat(componentName, "\" has an empty transformation output"));
                }
                (0, bui_assets_fs_1.saveIconComponent)({
                    assetCode: componentCode,
                    assetName: componentName,
                    targetDir: path_1.default.resolve(__dirname, packageSetName),
                    typeDeclarationCode: typeDeclarationCode,
                });
                assetNames.push(componentName);
                return [2 /*return*/];
            });
        });
    },
    onSetStart: function () {
        assetNames = [];
    },
    onSetComplete: function (_a) {
        var packageSetName = _a.packageSetName;
        var targetDir = path_1.default.resolve(__dirname, packageSetName);
        (0, bui_assets_fs_1.generateIndexFile)(targetDir, assetNames);
    },
});
