// @bookingcom/bui-assets-react/transport/TransportTaxiTypeExecElectric.js
import * as React from "react";
function TransportTaxiTypeExecElectric() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#7ED84E",
    fillRule: "nonzero",
    d: "M-18.851,-18.851C-8.44,-18.851 0,-10.411 0,0C0,10.411 -8.44,18.851 -18.851,18.851C-29.262,18.851 -37.702,10.411 -37.702,0C-37.702,-10.411 -29.262,-18.851 -18.851,-18.851",
    transform: "matrix(0 -1 -1 0 96.186 26.063)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,35.567C0,36.382 8.328,16.219 8.328,16.219L-5.294,15.913C-5.294,15.913 3.769,0.543 3.769,-0.815L10.829,-0.815L2.251,11.379L15.291,12.42L0,35.567Z",
    transform: "translate(90.908 31.529)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-52.444,-50.908C-57.64,-50.909 -61.852,-27.418 -61.852,1.558C-61.852,30.535 -57.64,54.026 -52.444,54.026C-47.249,54.027 -43.036,30.536 -43.036,1.56C-43.036,-27.417 -47.248,-50.908 -52.444,-50.908",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(88.297 48.21 15.476)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-2.809,-16.233C-1.076,-14.498 1.756,-12.066 1.424,-9.252C1.094,-6.438 -1.186,-0.168 -2.809,0.443L-10.012,-2.204L-2.809,-16.233Z",
    transform: "scale(-1 1) rotate(-21.641 203.607 57.48)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-5.413,-20.793C-3.748,-18.993 -0.368,-12.044 -0.808,-9.245C-1.247,-6.446 -3.767,-0.27 -5.413,0.28L-12.024,-2.231L-5.413,-20.793Z",
    transform: "scale(-1 1) rotate(-23.854 170.84 160.837)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,10.44C0.305,11.112 2.137,14.103 4.335,14.775C6.532,15.446 10.317,15.08 11.783,14.164C13.248,13.248 14.53,12.516 15.507,10.013C16.483,7.51 17.888,1.832 14.53,-0.854C11.172,-3.54 2.93,-5.006 2.076,-4.029C1.221,-3.052 0,10.44 0,10.44",
    transform: "translate(94.825 73.169)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-19.018C-1.097,-18.389 -2.316,-17.86 -3.298,-15.965C-4.281,-14.071 -4.772,-10.597 -4.702,-9.018C-4.632,-7.439 -4.105,-4.316 -2.772,-2.211C-1.439,-0.106 -0.281,0.877 1.404,1.122C3.088,1.368 7.322,1.014 8.059,0.839C8.796,0.664 9.813,0.032 10.305,-0.564C10.796,-1.161 11.884,-2.354 12.48,-5.091C13.077,-7.828 13.221,-12.396 10.343,-17.484C8.847,-20.129 2.386,-20.386 0,-19.018",
    transform: "translate(60.306 88.944)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.125,-2.108C-4.734,-2.129 -6.849,-1.195 -6.848,-0.021C-6.849,1.153 -4.734,2.121 -2.125,2.143C0.484,2.164 2.599,1.23 2.599,0.056C2.599,-1.118 0.484,-2.086 -2.125,-2.108",
    transform: "scale(1 -1) rotate(89.528 69.646 -7.984)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-16.545C-0.955,-15.997 -2.015,-15.538 -2.869,-13.889C-3.724,-12.241 -4.151,-9.219 -4.09,-7.845C-4.029,-6.472 -3.571,-3.755 -2.411,-1.924C-1.252,-0.092 -0.244,0.763 1.221,0.976C2.686,1.19 5.708,0.976 6.349,0.824C6.99,0.671 7.875,0.122 8.303,-0.397C8.73,-0.916 9.676,-1.954 10.195,-4.335C10.714,-6.716 11.05,-10.684 8.547,-15.11C7.246,-17.412 2.076,-17.735 0,-16.545",
    transform: "translate(13.506 85.014)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-3.056C0.901,-9.881 4.533,-9.3 7.752,-10.717C10.451,-11.904 13.576,-15.259 18.467,-17.239C28.513,-21.305 48.393,-19.935 61.224,-19.438C71.001,-19.059 84.033,-9.492 87.39,-6.209C87.39,-6.209 87.633,-6.245 90.213,-5.291C91.956,-4.647 95.366,-3.485 95.801,-3.445C96.237,-3.406 97.243,-3.445 99.122,-2.556C106.111,0.753 105.589,7.849 105.376,13.001C105.456,13.437 105.562,14.41 105.535,14.783C105.442,16.039 97.844,17.823 87.27,18.184C85.351,18.249 70.335,18.183 64.962,18.113C63.76,18.097 62.731,17.264 62.45,16.095C61.68,12.892 61.11,7.915 60.367,5.626C58.143,-1.225 49.031,0.605 48.342,8.284C47.899,13.221 48.532,15.246 49.102,17.208C40.991,16.795 22.901,15.436 14.421,14.423C13.799,14.349 12.627,13.635 12.389,12.447C11.352,7.527 13.369,0.621 7.418,-0.191C1.383,0.5 1.328,8.342 2.426,12.965C2.311,12.965 2.031,12.639 1.631,12.397C0.635,11.795 -0.041,10.333 -0.375,9.218C-0.821,7.733 -0.979,6.01 -0.927,4.565C-0.913,4.161 -0.852,3.759 -0.713,3.38C-0.078,1.648 -0.039,-2.177 0,-3.056",
    transform: "translate(7.712 68.326)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,5.141C-1.987,4.225 -0.997,3.127 -0.388,2.636C1.375,1.216 5.169,-1.933 10.928,-2.883C15.052,-3.563 20.428,-3.4 24.161,-0.431C28.201,2.783 31.665,7.59 32.142,8.442C30.347,8.704 5.104,7.495 0,5.141",
    transform: "translate(21.448 53.736)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BDBDBD",
    fillRule: "nonzero",
    d: "M0,-12.22C0,-12.22 5.038,-9.568 9.018,-5.829C11.803,-3.213 14.722,-0.736 15.093,-0.513C15.465,-0.29 23.783,0.091 27.583,0.014C34.649,-0.129 46.425,-0.633 45.639,-0.577C45.639,-0.577 41.433,-3.683 35.817,-7.102C30.772,-10.173 29.318,-10.675 25.301,-11.493C21.284,-12.311 0,-12.22 0,-12.22",
    transform: "translate(45.642 62.728)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-8.719C-2.407,-9.209 -11.01,-9.373 -17.522,-9.425C-15.867,-8.329 -13.885,-6.881 -12.11,-5.213C-9.325,-2.598 -6.406,-0.121 -6.035,0.102C-5.663,0.325 2.655,0.706 6.455,0.629C9.745,0.563 14.047,0.418 17.636,0.288C15.876,-0.923 13.382,-2.583 10.516,-4.328C5.471,-7.399 4.017,-7.901 0,-8.719",
    transform: "translate(66.77 62.113)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-6.9C1.069,-3.769 5.002,-0.943 5.511,-0.814C6.992,-0.439 11.396,0.055 13.702,-0.169C14.517,-0.247 16.903,-0.483 17.27,-0.851C15.36,-1.832 9.214,-4.823 3.05,-6.379C0.769,-6.955 0,-6.9 0,-6.9",
    transform: "translate(69.01 72.442)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-5.352C0.581,-4.868 1.327,-4.601 2.069,-3.321C2.8,-2.059 2.803,-0.547 3.263,-0.252C3.723,0.043 5.34,-0.158 5.715,-0.283C6.509,-0.548 6.13,-1.273 6.13,-1.273C6.13,-1.273 5.417,-3.027 4.185,-3.878C3.404,-4.417 0.477,-5.395 0,-5.352",
    transform: "translate(103.862 71.794)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-4.555C1.894,-4.464 9.134,-4.368 13.853,-4.968C14.377,-5.035 15.064,-4.829 15.385,-4.409C15.538,-4.21 15.664,-3.961 15.707,-3.656C15.845,-2.681 16.121,-0.793 16.121,-0.793C16.121,-0.793 11.009,0.208 7.346,0.269C3.684,0.329 -1.687,0.48 -3.952,0.151C-3.781,-0.584 -3.514,-2.877 -3.218,-3.464C-2.627,-4.633 -1.342,-4.619 0,-4.555",
    transform: "translate(92.952 80.683)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-1.705,-1.705C-4.092,-1.705 -6.025,-0.941 -6.025,-0C-6.025,0.942 -4.092,1.705 -1.705,1.705C0.68,1.705 2.614,0.942 2.614,-0C2.614,-0.941 0.68,-1.705 -1.705,-1.705",
    transform: "matrix(0 -1 -1 0 13.543 75.315)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-11.861C-0.437,-11.184 -0.858,-10.683 -2.138,-7.902C-4.265,-3.282 -4.309,-0.484 -4.309,-0.484L-0.732,0C-0.732,0 -0.238,-2.809 0.228,-4.361C0.695,-5.914 2.153,-10.16 3.549,-11.856L0,-11.861Z",
    transform: "translate(35.151 61.846)"
  }));
}
export default TransportTaxiTypeExecElectric;
