// @bookingcom/bui-assets-react/transport/TransportTaxiTypeMotorbike.js
import * as React from "react";
function TransportTaxiTypeMotorbike() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinecap: "round",
    strokeLinejoin: "round",
    strokeMiterlimit: 10,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-47.381,-47.717C-51.641,-47.596 -53.552,-27.643 -53.912,-1.073C-54.271,25.497 -50.442,48.236 -47.381,48.278C-44.32,48.319 -42.249,27.299 -41.889,0.73C-41.529,-25.84 -43.269,-47.833 -47.381,-47.717",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(89.151 63.156 3.568)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#454747",
    fillRule: "nonzero",
    d: "M0,-10.472C0.734,-8.082 3.377,-6.54 2.59,-4.072C2.057,-2.399 1.913,-0.328 3.301,1.11C4.883,2.75 5.885,0.253 6.784,-1.066C7.432,-2.017 7.977,-2.714 7.754,-3.749C7.544,-4.724 6.89,-5.762 6.37,-6.602C5.284,-8.36 5.314,-11.687 3.572,-13.222C2.913,-12.79 2.182,-12.511 1.542,-12.014C1.026,-11.613 0.591,-10.724 0,-10.472",
    transform: "translate(14.496 57.2)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-27.037C-6.48,-24.89 -9.022,-17.14 -12.365,-12.003C-15.359,-7.401 -13.513,-6.489 -11.075,-1.409C-10.914,-1.073 -10.657,-0.574 -10.345,0C-8.204,-0.9 -6.111,-1.854 -4.312,-3.148C-5.394,-5.317 -5.64,-6.356 -5.894,-7.422C-2.498,-14.695 6.049,-26.176 0,-27.037",
    transform: "translate(24.7 48.678)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M0,-16.488C-0.092,-17.405 0.794,-18.124 1.665,-17.822C3.92,-17.04 5.566,-15.746 7.204,-13.696C9.146,-11.265 10.997,-7.652 13.174,-6.062C18.811,-1.663 27.131,-9.804 27.562,-0.287C25.639,1.636 23.745,0.985 20.809,0.887C9.862,0.522 6.29,-0.448 2.506,-8.704C1.516,-10.864 0.331,-13.181 0,-16.488",
    transform: "translate(51.042 46.504)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#3A3A3A",
    fillRule: "nonzero",
    d: "M0,4.723C-0.593,3.943 -1.329,1.109 -0.542,0.422C0.031,-0.078 2.785,-0.416 3.497,-0.292C4.418,-0.131 6.545,1.573 6.721,2.484C6.812,2.955 6.164,4.072 5.929,4.433C5.523,5.059 5.421,5.139 4.604,4.984C3.834,4.838 3.555,4.804 2.781,4.922C1.868,5.061 0.807,4.442 0,4.723",
    transform: "translate(75.45 41.275)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M0,2.309C0.126,2.792 0.367,3.23 0.482,3.674C-1.318,4.448 -3.812,4.41 -5.771,4.909C-7.514,5.352 -8.127,6.323 -9.221,4.922C-10.12,3.769 -10.921,1.754 -12.075,0.874C-12.923,0.228 -13.451,-0.161 -13.353,-1.2C-13.265,-2.144 -12.608,-4.014 -11.378,-3.825C-11.378,-3.825 -2.968,-3.924 -1.781,-3.232C-0.594,-2.539 0,2.309 0,2.309",
    transform: "translate(81.3 47.46)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M0,1.427C-1.398,-0.123 -3.742,-5.611 -0.547,-5.769C1.304,-5.86 4.552,-6.053 6.536,-6.154C8.378,-6.248 10.554,-5.811 12.126,-4.95C13.885,-3.986 14.949,-2.557 15.884,-0.854C17.21,1.559 21.142,7.42 17.426,7.675",
    transform: "translate(75.715 49.531)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#515151",
    fillRule: "nonzero",
    d: "M-12.889,-12.78C-20.803,-12.878 -27.218,-7.186 -27.217,-0.068C-27.216,7.052 -20.802,12.901 -12.889,12.998C-4.976,13.097 1.439,7.405 1.438,0.286C1.438,-6.833 -4.976,-12.683 -12.889,-12.78",
    transform: "scale(1 -1) rotate(89.513 52.896 -23.89)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M-6.554,-6.554C-10.857,-6.554 -14.344,-3.62 -14.344,0C-14.344,3.619 -10.857,6.554 -6.554,6.554C-2.251,6.554 1.236,3.619 1.236,0C1.236,-3.62 -2.251,-6.554 -6.554,-6.554",
    transform: "matrix(0 -1 -1 0 27.166 82.268)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#969393",
    fillRule: "nonzero",
    d: "M0,-25.407C-2.283,-24.494 -4.841,-24.128 -3.836,-21.479C-2.831,-18.831 10.412,-0.655 10.412,0.259C10.412,1.172 16.167,3.456 17.08,2.634C22.197,-5.435 27.894,-3.366 28.407,-7.139C29.086,-12.59 25.773,-18.598 21.107,-22.94C16.027,-27.667 8.586,-28.863 2.146,-26.278C0.897,-25.777 0,-25.407 0,-25.407",
    transform: "translate(72.497 76.636)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-22.661C-1.866,-23.429 -2.261,-24.036 -3.908,-22.17C-5.554,-20.304 -6.695,-19.149 -6.256,-18.161C-5.817,-17.173 -4.28,-16.405 -4.061,-15.746C-3.841,-15.088 14.158,-1.039 16.463,0.168C18.768,1.375 26.78,-7.186 27,-8.064C27.219,-8.942 22.17,-13.551 21.073,-14.21C19.975,-14.868 16.244,-16.625 15.475,-16.734C14.707,-16.844 0,-22.661 0,-22.661",
    transform: "translate(22.852 83.292)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#494949",
    fillRule: "nonzero",
    d: "M0,-7.284C-0.233,-4.436 4.11,-3.597 6.375,-2.561C7.329,-2.125 7.36,-0.695 6.415,-0.269C4.716,0.498 0.966,0.277 -0.425,0.072C-1.663,-0.111 -4.804,0.108 -5.878,-0.42C-7.394,-1.166 -6.682,-3.041 -6.542,-4.425C-6.202,-7.782 -4.161,-6.619 0,-7.284",
    transform: "translate(66.936 108.65)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M0,-23.975C1.417,-18.109 -0.047,-2.402 0.388,4.559C-0.691,5.344 -1.266,6.125 -3.18,6.212C-5.143,6.301 -5.686,6.215 -7.443,5.516C-7.343,2.631 -8.325,-1.041 -8.749,-3.794C-9.191,-6.666 -9.043,-18.312 -8.533,-21.245C-8.221,-23.044 -8.672,-28.362 -7.1,-29.116C-4.687,-30.276 -1.519,-26.007 0,-23.975",
    transform: "translate(67.54 99.024)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M-2.452,-12.47C-0.185,-11.098 0.782,-2.73 -0.306,1.731C-0.571,2.818 -2.139,2.84 -2.452,1.765C-3.728,-2.602 -5.811,-7.405 -8.994,-10.74C-7.573,-11.629 -3.599,-12.886 -2.452,-12.47",
    transform: "scale(-1 1) rotate(-37.639 107.27 62.037)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#969393",
    fillRule: "nonzero",
    d: "M0,-25.331C14.27,1.358 -13.035,-3.272 -17.456,-5.987C-19.679,-7.352 -9.045,-19.003 -24.548,-21.747C-29.283,-22.584 -54.148,-33.346 -57.189,-32.585C-60.231,-31.825 -61.173,-28.605 -61.391,-27.41C-59.016,-25.279 -42.687,-10.124 -42.579,-10.124C-42.47,-10.124 -41.275,5.191 -37.365,5.625C-33.455,6.06 0.978,8.015 3.367,6.929C5.757,5.843 17.922,-15.881 18.9,-17.728C19.877,-19.574 19.986,-23.159 19.769,-23.81C19.551,-24.462 0,-25.331 0,-25.331",
    transform: "translate(79.422 93.95)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-25.89C1.037,-23.068 -0.068,-21.569 2.021,-19.407C5.253,-16.062 7.819,-17.136 11.572,-14.605C13.791,-13.109 16.84,-6.961 14.815,-1.633C14.409,-0.564 18.125,-1.485 17.209,-0.791C16.165,0 14.574,-0.929 13.385,-1.582C10.814,-2.994 9.086,-5.368 6.53,-6.85C3.791,-8.437 0.999,-9.649 -1.906,-10.888C-4.264,-11.892 -4.542,-13.272 -4.464,-15.895C-4.408,-17.793 -3.983,-19.827 -3.241,-21.558C-2.772,-22.652 -0.501,-23.494 0,-25.89",
    transform: "translate(36.27 67.144)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M0,-23.996C0.33,-16.987 14.743,-11.384 13.029,0C13.324,-1.963 5.946,-7.531 4.592,-8.964C2.144,-11.549 -1.857,-13.161 -4.724,-15.143C-6.114,-16.456 -4.109,-17.648 -4.305,-19.623C-4.369,-20.274 -4.759,-22.385 -4.336,-22.913C-3.917,-23.437 -0.405,-22.92 0,-23.996",
    transform: "translate(54.892 77.895)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-21.962C-0.973,-21.948 -1.504,-20.819 -0.888,-20.066C0.553,-18.303 4.214,-16.214 4.813,-15.39C5.691,-14.183 34.749,-3.754 38.787,-2.03C43.602,0.026 46.151,-0.778 45.157,-3.311C43.395,-7.802 35.601,-12.945 33.516,-14.371C31.431,-15.798 21.235,-20.176 15.695,-21.033C11.236,-21.723 1.824,-21.988 0,-21.962",
    transform: "translate(19.612 78.058)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "none",
    fillRule: "nonzero",
    stroke: "#E0E0E0",
    strokeWidth: 2,
    d: "M1.167,-0.564C0.835,0.559 -0.396,3.13 -0.918,4.139C-1.627,5.504 -0.091,7.174 1.167,8.005C4.122,9.956 5.758,12.362 6.583,10.335C7.117,9.023 8.238,6.856 8.879,5.867",
    transform: "scale(1 -1) rotate(-10.747 -321.76 -122.403)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-4.936C0.244,-4.062 1.739,1.326 3.037,-0.191C3.817,-1.102 2.714,-5.601 1.808,-6.262C1.348,-6.013 0.904,-5.758 0,-4.936",
    transform: "translate(18.952 57.028)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#ADADAD",
    fillRule: "nonzero",
    d: "M0,-7.696C-0.917,-5.049 0.117,-5.529 1.33,-3.061C1.875,-1.953 5.173,1.492 2.925,1.325C1.171,1.194 -2.423,-0.337 -3.853,-1.2C-4.882,-1.822 -6.969,-2.879 -7.441,-4.118C-8.051,-5.719 -5.087,-8.542 -3.72,-9.188C-2.659,-8.21 -2.256,-7.727 0,-7.696",
    transform: "translate(32.24 92.003)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-40.806C-0.481,-36.574 -2.136,-36.878 -1.566,-31.608C-1.008,-26.451 5.089,-25.932 6.654,-25.149C10.009,-23.472 9.035,-24.173 10.177,-21.235C8.511,-17.295 4.569,-4.993 2.773,-0.913C4.242,1.209 4.466,1.958 8.22,1.468C10.373,-2.838 12.783,-7.16 13.896,-9.688C15.434,-13.183 17.763,-15.134 18.593,-18.886C19.447,-22.752 21.63,-24.723 18.788,-27.302C9.003,-36.182 9.968,-38.161 12.134,-40.806C13.738,-42.764 11.866,-40.885 9.198,-41.197C6.03,-41.568 4.424,-39.748 0,-40.806",
    transform: "translate(23.237 85.579)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#454747",
    fillRule: "nonzero",
    d: "M0,-5.714C0.102,-4.598 0.087,-3.641 0.474,-2.527C0.86,-1.415 1.515,-0.411 2.28,0.474C3.23,1.574 4.859,2.463 6.321,2.026C7.582,1.649 8.335,0.094 8.749,-1.091C9.281,-2.609 10.321,-5.415 9.326,-6.93C8.507,-8.177 5.873,-7.917 4.615,-7.807C3.657,-7.723 2.973,-7.43 2.078,-7.112C1.015,-6.734 0.863,-6.196 0,-5.714",
    transform: "translate(29.448 17.276)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-28.783C-1.426,-27.751 -0.892,-27.504 -3.589,-26.423C-6.217,-25.37 -6.515,-24.406 -7.022,-22.678C-7.442,-21.245 -7.311,-10.572 -7.141,-8.702C-6.866,-5.68 -7.806,-1.93 -7.616,-1.899C-5.016,-1.472 12.002,0 12.793,-4.272C12.33,-8.855 12.045,-11.182 12.438,-14.132C12.678,-15.928 12.509,-18.437 12.341,-20.125C12.218,-21.351 11.801,-22.527 11.121,-23.554C10.566,-24.391 9.92,-25.278 9.573,-25.452C8.884,-25.796 5.949,-27.111 3.736,-27.701C1.524,-28.291 0,-28.783 0,-28.783",
    transform: "translate(28.995 47.52)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#454747",
    fillRule: "nonzero",
    d: "M3.243,-14.086C4.327,-12.387 3.129,-12.635 6.293,-7.916C6.27,-6.237 4.351,-3.754 3.243,-2.528C0.506,0.345 1.271,-7.14 -1.258,-9.695C-2.787,-11.24 0.935,-12.233 3.243,-14.086",
    transform: "scale(-1 1) rotate(8.899 -199.005 -213.593)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#454747",
    fillRule: "nonzero",
    d: "M0,5.229C-0.709,5.028 -1.186,3.961 -1.437,3.345C-1.701,2.696 -1.62,1.978 -1.178,1.404C-0.534,0.569 0.833,0.195 1.808,0.001C3.028,-0.241 4.641,-0.216 5.862,0.001C6.941,0.193 8.019,0.203 8.864,0.992C9.454,1.542 9.77,2.239 9.642,3.042C9.535,3.718 9.477,4.239 8.81,4C7.988,3.705 7.197,3.376 6.329,3.215C4.892,2.948 3.242,3.098 1.907,3.704C1.379,3.943 0.809,4.256 0.368,4.639C0.122,4.853 -0.182,5.47 0,5.229",
    transform: "translate(29.822 8.101)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M0,-3.922C-0.634,-2.609 -0.482,-1.17 -0.138,0.688C0.206,2.546 1.514,4.06 1.995,4.541C2.477,5.023 3.507,4.541 3.783,3.991C4.058,3.441 4.098,2.971 4.086,2.293C4.07,1.425 3.638,0.46 2.736,-0.233C1.827,-0.93 2.254,-2.128 2.736,-2.541C3.218,-2.954 4.812,-4.191 8.528,-3.847C12.243,-3.503 12.454,-3.096 12.73,-2.339C13.005,-1.583 12.253,-1.051 12.046,-0.432C12.166,0.447 12.025,1.66 11.687,3.481C11.485,4.572 12.179,4.748 12.661,4.197C13.142,3.647 12.644,0.821 13.204,-0.112C14.042,-1.51 14.175,-3.096 14.037,-3.784C13.899,-4.472 12.454,-8.945 7.5,-8.807C2.546,-8.67 0.963,-5.917 0,-3.922",
    transform: "translate(26.215 12.344)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M0,-1.826C-1.122,-1.029 0.221,1.845 1.784,1.157C5.405,-0.436 9.832,-0.759 13.614,0.419C14.088,0.567 14.555,0.491 13.976,-0.486C13.698,-0.954 13.197,-1.664 12.708,-2.152C11.912,-2.948 11.151,-3.308 8.979,-3.455C5.798,-3.671 1.955,-3.021 0,-1.826",
    transform: "translate(28.371 11.254)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-6.275C2.453,-6.002 2.206,0.72 2.942,1.145C3.544,1.494 43.854,6.704 44.703,5.855C45.553,5.005 52.106,-9.953 65.175,-19.659C65.463,-18.505 64.345,-10.861 64.345,-10.861C64.345,-10.861 58.256,9.501 49.882,11.443C41.508,13.384 2.5,11.308 1.772,10.216C1.044,9.124 0,-6.275 0,-6.275",
    transform: "translate(35.762 89.344)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#515151",
    fillRule: "nonzero",
    d: "M0,0.795C0,-2.431 1.18,-9.593 7.318,-13.055C13.89,-16.762 24.159,-14.708 27.307,-6.445C30.454,1.818 26.598,8.979 24.002,11.497C21.405,14.016 13.299,17.557 6.059,12.048C0.934,8.148 0,3.188 0,0.795",
    transform: "translate(84.724 96.742)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#666565",
    fillRule: "nonzero",
    d: "M0,-7.753C2.107,-9.282 4.338,-9.283 6.15,-8.81C10.922,-7.564 13.633,-4.667 15.436,-1.654C15.918,-0.848 16.386,1.53 13.769,-0.568C13.093,-1.11 9.609,-3.813 5.862,-4.39C3.566,-4.743 0.192,-4.87 -4.997,-1.507C-5.093,-2.564 -3.535,-5.187 0,-7.753",
    transform: "translate(94.787 86.76)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M6.213,-14.864C7.535,-13.839 12.107,-11.884 15.061,-6.817L6.213,3.802C6.213,3.802 -0.086,-2.809 -1.462,-3.082C-2.839,-3.354 6.213,-14.864 6.213,-14.864",
    transform: "scale(-1 1) rotate(52.228 -85.323 -25.576)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M0,-31.055C-1.575,-29.914 -0.986,-29.642 -3.965,-28.448C-6.868,-27.284 -7.197,-26.22 -7.757,-24.311C-8.221,-22.728 -6.167,-13.409 -5.979,-11.342C-5.676,-8.005 -5.154,-4.198 -5.709,-0.905C-5.745,-0.696 -5.609,-0.506 -5.399,-0.472C-2.528,0 14.291,-0.23 13.985,-0.945C13.473,-6.008 13.039,-13.278 13.474,-16.536C13.738,-18.52 14.069,-21.213 13.883,-23.078C13.748,-24.432 13.037,-24.144 12.285,-25.278C11.672,-26.203 10.958,-27.183 10.575,-27.375C9.815,-27.755 6.572,-29.208 4.128,-29.86C1.684,-30.512 0,-31.055 0,-31.055",
    transform: "translate(41.505 55.583)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#494949",
    fillRule: "nonzero",
    d: "M0,-10.236C2.151,-11.826 5.115,-10.087 4.34,-7.675C3.281,-4.38 6.052,-3.91 7.961,-2.398C10.093,-0.711 10.915,0.33 7.622,1.133C5.748,1.59 3.844,1.577 2.201,0.774C0.446,-0.083 -0.931,-1.507 -2.666,-2.254C-3.32,-2.535 -4.026,-3.652 -4.042,-4.364C-4.074,-5.816 -3.317,-6.123 -3.524,-8.089C-3.797,-10.675 -1.734,-9.344 0,-10.236",
    transform: "translate(47.806 112.972)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BCBBBB",
    fillRule: "nonzero",
    d: "M0,-48.356C-0.978,-46.184 -1.144,-45.456 -0.166,-42.632C0.812,-39.808 6.463,-28.75 6.463,-28.424C6.463,-28.098 5.757,-21.852 5.431,-16.312C5.105,-10.773 5.485,-2.355 5.485,-2.355C5.485,-2.355 11.703,0 14.447,-1.866C14.721,-3.238 16.728,-27.718 16.728,-30.542C16.728,-33.366 13.645,-41.86 13.904,-42.599C14.664,-44.771 17.488,-44.337 17.488,-45.858C17.488,-47.378 12.6,-47.596 12.6,-47.596L0,-48.356Z",
    transform: "translate(38.573 107.372)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M2.321,-11.666C3.899,-10.467 5.887,-10.1 6.315,-9.012C6.744,-7.924 6.477,-0.818 2.321,-0.259C1.484,-0.514 -1.983,-3.879 -1.671,-4.024C-1.359,-4.168 2.321,-11.666 2.321,-11.666",
    transform: "scale(-1 1) rotate(43.298 -60.629 -51.062)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M-6.662,-9.986C-7.745,-9.907 -8.791,-6.31 -9.004,-4.363C-9.216,-2.417 -9.134,0.993 -6.662,0.615C-4.19,0.237 -0.174,-0.68 1.184,-4.004C2.541,-7.329 -1.663,-10.355 -6.662,-9.986",
    transform: "scale(-1 1) rotate(-25.68 28.643 101.526)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,5.777C-0.783,5.555 -1.31,4.377 -1.587,3.695C-1.879,2.978 -1.789,2.185 -1.301,1.552C-0.59,0.629 0.92,0.216 1.998,0.002C3.345,-0.266 5.127,-0.238 6.476,0.002C7.668,0.213 8.858,0.225 9.792,1.096C10.444,1.704 10.793,2.473 10.652,3.361C10.533,4.108 10.469,4.684 9.733,4.419C8.825,4.093 7.95,3.73 6.992,3.552C5.404,3.257 3.581,3.423 2.106,4.092C1.523,4.357 0.894,4.702 0.407,5.125C0.134,5.361 -0.201,6.043 0,5.777",
    transform: "translate(42.419 12.78)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-4.333C-0.7,-2.882 -0.532,-1.292 -0.152,0.76C0.228,2.812 1.672,4.485 2.204,5.017C2.736,5.549 3.875,5.017 4.179,4.409C4.483,3.8 4.527,3.281 4.514,2.533C4.496,1.574 4.019,0.508 3.022,-0.258C2.019,-1.028 2.49,-2.351 3.022,-2.807C3.555,-3.263 5.316,-4.63 9.421,-4.25C13.525,-3.87 13.758,-3.421 14.062,-2.585C14.366,-1.749 13.536,-1.161 13.308,-0.477C13.44,0.493 13.284,1.833 12.911,3.845C12.687,5.05 13.454,5.245 13.986,4.637C14.518,4.028 13.968,0.906 14.587,-0.124C15.513,-1.668 15.659,-3.421 15.507,-4.181C15.355,-4.941 13.758,-9.882 8.285,-9.73C2.812,-9.578 1.064,-6.537 0,-4.333",
    transform: "translate(38.434 17.467)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-2.062C-1.267,-1.163 0.25,2.082 2.015,1.306C6.102,-0.493 11.1,-0.858 15.369,0.472C15.905,0.639 16.432,0.554 15.778,-0.55C15.465,-1.078 14.899,-1.879 14.347,-2.43C13.448,-3.329 12.589,-3.736 10.137,-3.901C6.545,-4.144 2.207,-3.411 0,-2.062",
    transform: "translate(40.569 15.108)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#D3D1D1",
    fillRule: "nonzero",
    d: "M0,9.431C-3.503,8.774 -7.571,8.612 -9.875,7.558C-14.429,6.523 -14.769,2.106 -16.975,-1.251C-19.043,-4.399 -21.835,-5.118 -23.574,-1.623C-24.002,-0.762 -23.445,1.118 -23.299,2.025C-22.207,8.85 -19.277,12.514 -15.346,13.482C-14.175,13.77 -7.401,14.549 -1.901,14.314C-2.05,12.358 -1.317,10.672 0,9.431",
    transform: "translate(58.004 33.234)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M0,-3.895C-4.13,-3.761 -5.985,-3.684 -7.508,-3.506C-8.335,-3.41 -8.355,-1.697 -7.611,-1.408C-6.868,-1.119 4.187,0 5.14,-0.542C6.092,-1.084 6.318,-3.703 5.063,-3.703C3.058,-3.703 0,-3.895 0,-3.895",
    transform: "translate(64.01 48.144)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#3A3A3A",
    fillRule: "nonzero",
    d: "M0,-4.861C1.224,-5.369 2.169,-6.305 3.533,-6.073C4.343,-5.935 5.111,-5.516 5.861,-5.21C6.4,-4.99 6.781,-4.516 6.896,-3.946C7.063,-3.119 7.149,-2.27 7.172,-1.426C7.187,-0.911 7.238,-0.35 6.871,0.04C6.301,0.644 6.153,0.09 5.551,0.18C4.601,0.321 3.655,1.444 2.503,0.725C2.331,0.617 2.221,0.472 2.137,0.311C1.791,-0.352 0.897,-0.862 0.15,-0.826C-0.778,-0.78 -1.02,-0.584 -1.313,-1.422C-1.743,-2.651 -0.819,-4.49 0,-4.861",
    transform: "translate(56.944 47.672)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M-7.646,-7.646C-12.666,-7.646 -16.735,-4.223 -16.735,0C-16.735,4.223 -12.666,7.646 -7.646,7.646C-2.626,7.646 1.443,4.223 1.443,0C1.443,-4.223 -2.626,-7.646 -7.646,-7.646",
    transform: "matrix(0 -1 -1 0 97.857 89.544)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M-2.435,-2.263C-4.483,-2.263 -6.143,-1.169 -6.143,0.179C-6.143,1.528 -4.483,2.62 -2.435,2.621C-0.387,2.621 1.273,1.527 1.273,0.179C1.273,-1.17 -0.387,-2.262 -2.435,-2.263",
    transform: "scale(1 -1) rotate(85.796 50.278 20.793)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "none",
    fillRule: "nonzero",
    stroke: "#6D6C6C",
    strokeLinecap: "butt",
    strokeLinejoin: "miter",
    strokeWidth: 1,
    d: "M-3.398,-5.628C-3.081,-5.679 9.319,-5.628 9.319,-5.628",
    transform: "scale(1 -1) rotate(-62.252 8.82 -77.94)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#6D6C6C",
    fillRule: "nonzero",
    d: "M-2.074,-1.519C-4.122,-1.518 -5.782,-0.544 -5.782,0.657C-5.782,1.859 -4.122,2.833 -2.074,2.834C-0.027,2.833 1.633,1.859 1.633,0.657C1.633,-0.544 -0.027,-1.518 -2.074,-1.519",
    transform: "scale(1 -1) rotate(72.414 65.297 45.122)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "none",
    fillRule: "nonzero",
    stroke: "#6D6C6C",
    strokeLinecap: "butt",
    strokeLinejoin: "miter",
    strokeWidth: 1,
    d: "M-7.214,-6.301C-6.896,-6.353 5.504,-6.301 5.504,-6.301",
    transform: "scale(-1 1) rotate(82.273 -69.494 -33.255)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M2.702,-6.134C4.415,-4.605 9.84,3.394 8.703,4.33C7.535,4.35 4.14,2.103 2.702,1.312C1.028,0.391 -0.942,-1.277 -0.66,-1.116C-3.647,-3.169 0.992,-8.108 2.702,-6.134",
    transform: "scale(-1 1) rotate(74.611 -87.052 -36.342)"
  }));
}
export default TransportTaxiTypeMotorbike;
