// @bookingcom/bui-assets-react/transport/TransportTaxiTypeWatertaxi.js
import * as React from "react";
function TransportTaxiTypeWatertaxi() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M0,-10.912C-5.495,-12.585 -11.222,-13.068 -16.927,-12.625C-21.651,-12.259 -26.283,-13.86 -32.042,-16.179C-50.188,-22.351 -58.586,-14.973 -76.082,-11.292C-81.583,-10.134 -81.922,-5.751 -95.907,-10.078C-92.206,-7.593 -91.236,-3.462 -83.885,-1.63C-63.679,3.407 -65.99,-7.322 -49.462,-1.454C-39.887,2.433 -33.688,0.113 -27.564,-1.335C-11.345,-8.179 -1.05,11.439 19.64,-8.791C7.57,-5.358 5.197,-9.33 0,-10.912",
    opacity: 0.47,
    transform: "translate(95.41 91.973)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#D9D9D9",
    fillRule: "nonzero",
    d: "M0,-14.643C-3.921,-20.009 -6.093,-19.015 -11.409,-19.505C-11.409,-19.505 -13.198,-26.406 -14.626,-27.126C-16.053,-27.845 -26.733,-33.128 -32.378,-33.318C-43.414,-32.935 -80.555,-27.467 -90.79,-26.223C-91.516,-26.134 -91.783,-25.22 -91.218,-24.757C-90.031,-23.786 -88.682,-22.597 -87.525,-21.482C-88,-19.437 -88.764,-16.22 -88.764,-16.22C-88.764,-16.22 -100.309,-15.789 -101.36,-15.537C-101.732,-14.676 -101.508,12.218 -101.508,12.218C-99.579,12.439 -47.214,15.648 -39.606,16.637C-34.899,17.342 -26.374,18.675 -20.436,12.038C-16.284,7.396 -9.008,-3.027 -5,-8.886C-0.57,-10.409 2.694,-11.01 0,-14.643",
    transform: "translate(118.287 61.325)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-21.329C7.993,-22.833 46.73,-21.511 50.999,-24.371C56.391,-28.128 60.875,-30 63.29,-29.881C65.705,-29.762 95.892,-26.945 97.237,-26.919C98.583,-26.892 99.935,-26.955 100.238,-26.3C100.541,-25.645 103.503,-21.88 103.457,-21.345C103.41,-20.811 95.782,-8.297 94.127,-6.055C92.088,-3.292 84.668,6.794 76.23,7.77C68.45,8.671 22.467,6.626 11.062,6.603C7.72,6.597 2.679,1.482 2.365,-0.575C1.113,-8.771 -0.387,-21.18 0,-21.329",
    transform: "translate(10.587 73.466)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,12.933L1.816,5.878C1.816,5.878 39.647,0 40.302,0.603C40.068,3.346 40.554,5.275 40.371,6.573C39.702,6.469 27.862,7.908 24.052,11.52C18.783,11.768 0,12.933 0,12.933",
    transform: "translate(35.25 32.072)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-9.422C-0.383,-9.865 1.998,-2.469 1.998,-2.469L25.876,0.453C25.876,0.453 24.859,-2.635 23.895,-4.863C20.774,-6.089 14.078,-9.118 10.559,-9.494C7.002,-9.875 0,-9.422 0,-9.422",
    transform: "translate(79.152 41.56)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#474646",
    fillRule: "nonzero",
    d: "M0,0.488C-3.121,-0.738 -9.817,-3.767 -13.336,-4.144C-13.661,-4.178 -14.022,-4.204 -14.396,-4.226C-16.237,-4.16 -17.613,-4.071 -17.613,-4.071C-17.996,-4.514 -15.615,2.882 -15.615,2.882L1.709,5.002C1.345,3.941 0.664,2.022 0,0.488",
    transform: "translate(96.765 36.21)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-5.458C-0.057,-6.04 -0.487,-6.462 -1.005,-6.437C-2.299,-6.374 -3.093,-5.293 -4.305,-3.1C-5.756,-0.471 -4.651,1.004 -3.585,0.686C-2.52,0.367 -0.94,-2.221 -0.3,-3.481C0.034,-4.136 0.056,-4.888 0,-5.458",
    transform: "translate(100.937 55.598)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-3.599C0.218,-4.557 -0.253,-5.228 -0.942,-4.889C-1.638,-4.547 -2.31,-3.611 -3.294,-2.032C-4.762,0.323 -4.259,1.629 -3.466,1.337C-2.673,1.044 -1.123,-1.276 -0.458,-2.404C-0.23,-2.79 -0.088,-3.211 0,-3.599",
    transform: "translate(111.284 57.371)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#D9D9D9",
    fillRule: "nonzero",
    d: "M0,10.702C0.201,11.283 1.141,16.967 1.651,20.103C1.843,21.281 2.865,22.129 4.059,22.099C8.721,21.981 28.551,21.441 43.275,21.291C63.632,21.083 74.119,17.997 79.418,13.044C85.842,7.038 94.874,-11.427 94.874,-11.427L83.953,3.594C83.953,3.594 80.152,9.608 72.34,13.962C67.991,16.386 16.978,18.201 8.493,17.686C3.472,17.381 0,10.702 0,10.702",
    transform: "translate(12.952 62.19)"
  }));
}
export default TransportTaxiTypeWatertaxi;
