// @bookingcom/bui-assets-react/transport/TransportCompare.js
import * as React from "react";
function TransportCompare() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-1.954,-11.219C-0.75,-10.014 1.217,-8.323 0.987,-6.379C0.757,-4.435 -0.827,-0.106 -1.954,0.312L-6.957,-1.541L-1.954,-11.219Z",
    transform: "scale(-1 1) rotate(-21.768 122.595 145.284)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-3.763,-14.374C-2.607,-13.123 -0.259,-8.306 -0.565,-6.372C-0.87,-4.438 -2.62,-0.175 -3.763,0.199L-8.355,-1.559L-3.763,-14.374Z",
    transform: "scale(-1 1) rotate(-23.99 99.618 206.754)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,7.212C0.212,7.676 1.486,9.742 3.014,10.206C4.542,10.67 7.173,10.417 8.192,9.785C9.211,9.152 10.102,8.646 10.781,6.917C11.46,5.188 12.436,1.266 10.102,-0.59C7.767,-2.446 2.037,-3.458 1.443,-2.783C0.849,-2.108 0,7.212 0,7.212",
    transform: "translate(106.302 51.714)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-13.138C-0.763,-12.703 -1.61,-12.338 -2.293,-11.029C-2.976,-9.72 -3.318,-7.321 -3.269,-6.23C-3.22,-5.139 -2.854,-2.982 -1.927,-1.528C-1,-0.073 -0.195,0.605 0.976,0.775C2.147,0.945 5.091,0.7 5.603,0.579C6.115,0.458 6.823,0.022 7.164,-0.39C7.506,-0.802 8.262,-1.626 8.677,-3.517C9.092,-5.408 9.192,-8.564 7.191,-12.078C6.151,-13.906 1.659,-14.083 0,-13.138",
    transform: "translate(82.302 62.612)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-11.429C-0.664,-11.051 -1.401,-10.733 -1.995,-9.594C-2.589,-8.456 -2.886,-6.368 -2.844,-5.419C-2.801,-4.47 -2.483,-2.594 -1.677,-1.329C-0.87,-0.063 -0.17,0.527 0.849,0.675C1.868,0.822 3.969,0.675 4.414,0.569C4.86,0.464 5.475,0.084 5.773,-0.274C6.07,-0.633 6.728,-1.35 7.088,-2.994C7.449,-4.639 7.683,-7.38 5.942,-10.438C5.038,-12.028 1.443,-12.251 0,-11.429",
    transform: "translate(49.764 59.896)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#727272",
    fillRule: "nonzero",
    d: "M0,-2.112C0.627,-6.827 3.152,-6.425 5.39,-7.404C7.266,-8.224 9.439,-10.542 12.84,-11.909C19.824,-14.718 33.645,-13.772 42.566,-13.428C49.364,-13.167 58.425,-6.558 60.758,-4.29C60.758,-4.29 60.928,-4.315 62.721,-3.656C63.933,-3.211 66.304,-2.408 66.606,-2.381C66.909,-2.354 67.609,-2.381 68.915,-1.767C73.774,0.519 73.412,5.421 73.264,8.98C73.319,9.281 73.393,9.954 73.374,10.211C73.309,11.079 68.027,12.311 60.675,12.56C59.341,12.606 48.901,12.56 45.165,12.511C44.329,12.501 43.614,11.925 43.419,11.118C42.883,8.905 42.487,5.467 41.971,3.886C40.424,-0.847 34.089,0.417 33.61,5.722C33.302,9.132 33.742,10.531 34.138,11.886C28.5,11.601 15.922,10.662 10.026,9.963C9.594,9.911 8.779,9.418 8.614,8.598C7.893,5.199 9.295,0.428 5.158,-0.133C0.962,0.345 0.923,5.762 1.687,8.956C1.607,8.956 1.412,8.73 1.134,8.563C0.441,8.147 -0.028,7.137 -0.261,6.367C-0.571,5.341 -0.681,4.151 -0.645,3.152C-0.634,2.874 -0.592,2.596 -0.496,2.334C-0.054,1.137 -0.027,-1.505 0,-2.112",
    transform: "translate(45.736 48.37)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,3.551C-1.382,2.918 -0.693,2.16 -0.27,1.821C0.956,0.84 3.594,-1.336 7.598,-1.992C10.465,-2.462 14.203,-2.349 16.798,-0.298C19.607,1.922 22.015,5.243 22.347,5.832C21.099,6.013 3.549,5.177 0,3.551",
    transform: "translate(55.286 38.29)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-8.441C0,-8.441 3.502,-6.609 6.27,-4.026C8.206,-2.219 10.235,-0.508 10.494,-0.354C10.752,-0.2 16.536,0.063 19.177,0.01C24.09,-0.089 32.277,-0.437 31.731,-0.398C31.731,-0.398 28.807,-2.544 24.902,-4.905C21.394,-7.027 20.383,-7.374 17.591,-7.939C14.798,-8.504 0,-8.441 0,-8.441",
    transform: "translate(72.107 44.501)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-4.766C0.743,-2.603 3.477,-0.651 3.832,-0.562C4.861,-0.303 7.923,0.038 9.526,-0.116C10.093,-0.17 11.752,-0.333 12.007,-0.587C10.679,-1.265 6.406,-3.331 2.12,-4.406C0.535,-4.804 0,-4.766 0,-4.766",
    transform: "translate(88.353 51.211)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-3.697C0.404,-3.363 0.923,-3.178 1.439,-2.294C1.947,-1.422 1.949,-0.378 2.268,-0.174C2.588,0.03 3.713,-0.109 3.973,-0.195C4.525,-0.378 4.262,-0.879 4.262,-0.879C4.262,-0.879 3.766,-2.091 2.909,-2.679C2.366,-3.051 0.332,-3.727 0,-3.697",
    transform: "translate(112.585 50.764)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-3.146C1.317,-3.083 6.351,-3.017 9.632,-3.432C9.996,-3.478 10.474,-3.335 10.697,-3.045C10.803,-2.908 10.89,-2.735 10.92,-2.525C11.017,-1.852 11.208,-0.547 11.208,-0.547C11.208,-0.547 7.654,0.144 5.107,0.186C2.561,0.228 -1.173,0.332 -2.747,0.105C-2.629,-0.403 -2.443,-1.987 -2.237,-2.392C-1.826,-3.2 -0.933,-3.19 0,-3.146",
    transform: "translate(105 56.904)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M0,-14.713C-2.998,-13.667 -5.946,-10.662 -5.846,-8.05C-5.747,-5.439 -4.852,-0.678 -4.852,-0.678L2.948,0C2.948,0 3.748,-13.375 2.998,-13.987C2.249,-14.6 0,-14.713 0,-14.713",
    transform: "translate(47.473 91.037)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-15.699C-0.993,-15.52 -2.889,-14.892 -3.747,-12.2C-4.408,-10.126 -4.785,-7.535 -4.063,-4.889C-3.341,-2.243 -2.257,-0.807 -0.993,-0.403C0.271,0 3.521,-0.359 3.882,-0.583C4.244,-0.807 6.094,-2.377 6.275,-5.248C6.456,-8.119 5.959,-13.008 4.74,-14.264C3.521,-15.52 0,-15.699 0,-15.699",
    transform: "translate(10.246 92.707)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-16.058C-1.264,-15.475 -4.966,-12.918 -4.153,-5.786C-3.341,1.346 1.174,1.435 1.535,1.48C1.896,1.525 5.643,1.435 5.959,1.301C6.275,1.166 7.765,0.314 8.487,-0.807C9.209,-1.929 9.932,-2.781 10.067,-5.607C10.203,-8.433 10.383,-13.636 7.539,-15.609C4.695,-17.583 1.129,-16.686 0,-16.058",
    transform: "translate(47.716 94.411)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-3.582C0.273,-2.605 0.983,-1.194 1.857,-0.38C2.731,0.434 3.223,1.303 5.462,1.465C7.702,1.628 9.723,1.031 10.214,0.76C10.706,0.488 11.744,0 12.509,-1.574C13.273,-3.148 13.383,-4.233 13.328,-4.722C13.273,-5.21 0,-3.582 0,-3.582",
    transform: "translate(72.223 92.864)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "silver",
    fillRule: "nonzero",
    d: "M0,-33.725C-0.586,-33.619 -1.106,-33.284 -1.416,-32.779C-3.107,-30.013 -6.178,-21.51 -6.654,-19.139C-6.898,-17.927 -6.407,-12.191 -6.531,-11.945C-6.654,-11.699 -6.956,-10.877 -6.956,-9.49C-6.956,-9.113 -6.666,-5.436 -6.527,-3.689C-6.483,-3.139 -6.102,-2.682 -5.566,-2.537C-4.278,-2.191 -1.793,-1.509 -1.486,-1.421C-1.315,-1.882 -2.193,-4.984 -0.095,-8.74C2.095,-12.662 4.979,-9.363 5.908,-6.596C6.836,-3.829 6.937,0.213 7.061,0.398C16.061,1.427 36.491,2.769 36.491,2.769C36.491,2.769 35.405,-7.816 40.484,-9.354C42.65,-9.661 45.181,-9.567 46.529,-5.056C47.583,-1.529 48.358,2.996 48.358,2.996C48.358,2.996 49.596,2.811 50.091,2.811C50.586,2.811 72.802,2.135 73.297,2.135C73.792,2.135 81.774,1.52 82.084,1.458C82.393,1.397 82.888,1.09 82.826,0.844C82.764,0.598 82.455,-0.14 82.517,-0.509C82.579,-0.878 82.932,-4.396 82.932,-4.704C82.886,-5.703 80.97,-12.499 80.846,-12.868C80.722,-13.236 79.443,-16.256 79.299,-16.618C78.725,-18.062 71.069,-23.197 70.388,-23.504C69.707,-23.812 59.188,-31.682 58.321,-32.297C57.455,-32.912 54.863,-34.514 52.964,-34.902C48.947,-35.724 43.013,-35.782 36.769,-36.123C25.818,-36.721 9.115,-35.377 0,-33.725",
    transform: "translate(7.364 87.922)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-10.473C0,-10.473 5.226,-5.221 6.424,-4.005C7.852,-2.556 9.407,-0.681 11.377,-0.256C13.347,0.17 32.308,-0.26 31.965,-0.515C31.622,-0.77 29.209,-2.953 26.297,-5.081C23.385,-7.208 20.301,-9.161 19.515,-9.367C18.171,-9.718 2.227,-10.643 0,-10.473",
    transform: "translate(41.696 65.288)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-9.764C-0.304,-8.491 -0.755,-6.599 -1.143,-0.1C-1.143,-0.1 8.467,0.148 11.438,0.245C14.361,0.245 13.931,-0.83 13.325,-1.782C12.9,-2.447 11.809,-4.439 8.954,-6.724C6.099,-9.009 4.725,-9.536 3.694,-9.773C2.663,-10.009 0,-9.764 0,-9.764",
    transform: "translate(30.751 65.076)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-7.542C-1.807,-7.297 -1.787,-6.318 -2.362,-5.094C-2.937,-3.869 -4.662,1.109 -4.662,1.109L3.966,1.258C3.966,1.258 10.227,1.258 11.86,1.225C13.481,1.192 14.738,0.83 15.366,-1.569C15.614,-2.513 16.009,-4.614 16.359,-6.565C16.623,-8.044 15.929,-8.8 14.275,-8.734C9.397,-8.541 1.538,-7.75 0,-7.542",
    transform: "translate(10.34 64.023)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-7.469C0.205,-6.385 0.75,-3.065 0.886,-2.794C1.023,-2.523 6.342,-1.507 6.819,-1.507C7.296,-1.507 15.086,0.339 15.086,0.068C15.086,-0.203 13.722,-3.726 13.381,-4.268C13.04,-4.81 12.29,-6.368 11.744,-6.571C11.199,-6.775 0.341,-7.808 0,-7.469",
    transform: "translate(53.329 76.76)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-6.436C0.136,-6.165 0.682,-3.997 0.682,-3.523C0.682,-3.048 0.955,0.271 0.955,0.271C0.955,0.271 3.341,-0.813 3.341,-1.219C3.341,-1.626 2.659,-3.794 2.318,-4.539C1.977,-5.284 1.5,-6.707 0,-6.436",
    transform: "translate(83.282 76.608)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,-5.66C0.15,-5.411 2.257,-0.428 2.257,-0.428C2.257,-0.428 7.271,0.269 8.174,0.169C9.076,0.07 11.684,-0.428 11.684,-0.428C11.684,-0.428 11.684,-4.614 11.032,-5.511C11.032,-5.511 9.596,-5.808 7.071,-5.859C3.648,-5.929 0.058,-5.66 0,-5.66",
    transform: "translate(69.213 76.709)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-8.252C-0.654,-8.209 -2.241,-8.116 -6.111,-7.905C-9.951,-7.695 -18.567,-7.905 -20.113,-7.995C-21.658,-8.085 -23.386,-8.176 -24.477,-7.453C-25.371,-6.861 -26.237,-5.454 -27.325,-3.803C-29.014,-1.312 -31.412,-0.381 -32.565,-0.328C-32.565,-0.328 -29.654,0.045 -29.386,0.045C-28.204,0.045 0.457,-0.391 1.372,-1.845C1.098,-2.391 0.78,-4.533 1.143,-5.617C1.418,-7.752 0.686,-8.297 0,-8.252",
    transform: "translate(88.284 91.243)"
  }));
}
export default TransportCompare;
