// @bookingcom/bui-assets-react/transport/TransportPublicBus.js
import * as React from "react";
function TransportPublicBus() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M0,-8.126C0.424,-8.183 4.811,-7.922 5.238,-7.903L101.147,-4.661C101.928,-4.542 101.921,-3.414 101.139,-3.305L76.513,0.131C74.258,0.446 71.966,0.375 69.735,-0.078C69.735,-0.078 6.724,-3.804 -8.693,-6.454C-10.926,-6.838 -10.815,-8.313 -7.579,-8.46C-5.127,-8.572 0,-8.126 0,-8.126",
    opacity: 0.47,
    transform: "translate(12.202 92.459)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-3.733C0.107,-3.318 0.28,-1.787 1.391,-0.893C2.503,0 3.144,-0.128 3.499,-0.128C3.854,-0.128 4.474,-0.351 4.474,-0.351C4.474,-0.351 5.77,-0.159 6.794,-0.447C7.818,-0.734 7.862,-0.606 8.4,-1.276C8.938,-1.946 9.536,-3.127 9.536,-3.127L0,-3.733Z",
    transform: "translate(30.33 86.09)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M6.49,-7.959C6.953,-7.59 8.583,-4.795 8.433,-3.122C8.283,-1.448 7.507,0.659 6.49,1.615C5.474,2.571 3.792,3.348 2.827,3.259C1.863,3.169 6.49,-7.959 6.49,-7.959",
    transform: "scale(-1 1) rotate(69.793 -102.695 -22.697)"
  }), /*#__PURE__*/React.createElement("use", {
    xlinkHref: "#40599vtcm3wtod0ka",
    width: 12.185,
    height: 14.49,
    x: 54.735,
    y: 72.887,
    transform: "scale(.93734 .966)"
  }), /*#__PURE__*/React.createElement("use", {
    xlinkHref: "#40599vtcm3wtod0lb",
    width: 7.191,
    height: 10.399,
    x: 10.28,
    y: 75.329,
    transform: "scale(.89886 .94536)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-16.06C-1.252,-15.651 -2.424,-14.886 -3.22,-13.257C-4.015,-11.628 -4.205,-8.787 -4.129,-6.894C-4.053,-5 -3.674,-3.333 -3.03,-2.348C-2.386,-1.363 -1.591,-0.303 -0.644,0.114C0.303,0.531 1.136,0.644 2.197,0.606C3.257,0.569 4.621,0.569 5.416,-0.037C6.212,-0.644 7.121,-1.25 7.689,-2.651C8.257,-4.053 9.204,-5.303 9.015,-7.386C8.826,-9.469 9.015,-12.878 7.235,-14.355C5.454,-15.833 1.97,-16.704 0,-16.06",
    transform: "translate(57.504 89.406)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E9E9E9",
    fillRule: "nonzero",
    d: "M0,35.232C-0.077,35.077 -0.541,34.072 -0.657,33.994C-0.773,33.917 -1.006,33.685 -1.006,33.685C-1.006,33.685 -0.155,32.641 -0.116,32.409C-0.077,32.177 -0.271,30.088 -0.271,30.088C-0.271,30.088 -0.812,24.713 -0.889,24.519C-0.967,24.326 -3.519,11.873 -3.829,10.906C-4.138,9.939 -5.84,5.453 -6.226,4.486C-6.613,3.519 -7.077,2.668 -7.425,2.359C-7.773,2.05 -8.585,1.276 -8.701,1.199C-8.817,1.121 -9.63,-0.039 -10.055,-0.464C-10.48,-0.89 -11.409,-1.547 -12.105,-1.857C-12.801,-2.166 -14.541,-2.785 -14.541,-2.785C-14.541,-2.785 -15.315,-3.945 -15.701,-4.486C-16.088,-5.028 -17.519,-6.497 -18.06,-6.884C-18.602,-7.271 -24.055,-8.741 -24.558,-8.857C-25.06,-8.973 -32.525,-10.52 -33.066,-10.52C-33.607,-10.52 -36.701,-10.713 -36.972,-10.558C-37.243,-10.404 -37.746,-10.133 -37.746,-10.133C-37.746,-10.133 -41.806,-10.404 -43.663,-10.288C-45.519,-10.172 -57.083,-8.045 -57.779,-7.813C-58.475,-7.581 -76.381,-2.592 -76.768,-2.437C-77.155,-2.282 -79.978,-0.658 -80.365,-0.387C-80.751,-0.117 -81.022,0.193 -81.061,0.347C-81.099,0.502 -81.138,0.734 -81.138,0.734C-81.138,0.734 -105.116,6.883 -105.928,7.154C-106.74,7.424 -108.867,7.734 -108.867,7.734C-113.145,8.9 -114.005,10.021 -114.63,12.375C-115.172,14.417 -115.459,23.86 -115.47,28.486C-114.517,28.451 -70.937,25.552 -68.669,25.228C-66.388,24.902 -59.544,24.142 -58.24,24.468C-56.937,24.794 -31.734,32.507 -31.734,32.507C-31.734,32.507 -31.743,43.958 -30.978,45.674C-30.823,45.712 -22.238,45.945 -21.967,45.945C-21.696,45.945 -4.873,45.133 -4.1,45.017C-3.326,44.9 -2.127,44.862 -1.547,44.475C-0.967,44.088 -0.464,43.547 -0.387,43.199C-0.309,42.851 0.193,40.105 0.193,39.486C0.193,38.867 0.271,38.558 0.271,38.558C0.271,38.558 0.077,35.387 0,35.232",
    transform: "translate(115.813 40.386)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-11.174C-0.386,-10.981 -1.174,-10.644 -1.553,-9.886C-1.932,-9.129 -2.235,-7.841 -2.311,-7.007C-2.386,-6.174 -2.273,-3.901 -1.856,-2.386C-1.439,-0.871 -1.061,-0.189 -0.644,0.227C-0.227,0.644 1.288,0.644 2.007,0.644C2.727,0.644 3.03,0.53 3.03,0.53C3.03,0.53 5,0.758 5.833,0.417C6.666,0.076 6.969,-0.227 7.31,-0.947C7.651,-1.667 7.992,-1.742 7.992,-3.409C7.992,-5.076 7.992,-9.204 6.742,-10.568C5.492,-11.932 1.288,-11.818 0,-11.174",
    transform: "translate(12.694 85.125)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-24.743C0,-24.743 3.222,-11.525 3.635,-8.799C4.048,-6.072 5.617,0.041 5.617,0.041C5.617,0.041 18.07,0.654 20.626,0.816C24.696,1.074 32.632,1.032 32.632,1.032C32.632,1.032 30.897,-9.129 30.236,-11.607C29.575,-14.086 27.51,-18.96 27.014,-19.951C26.519,-20.943 26.601,-21.686 24.536,-22.512C22.471,-23.338 16.192,-24.33 15.696,-24.412C15.201,-24.495 5.535,-25.817 4.378,-25.652C3.222,-25.486 0,-24.743 0,-24.743",
    transform: "translate(82.141 64.068)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#474646",
    fillRule: "nonzero",
    d: "M0,18.465C-0.386,16.41 -0.764,14.514 -1.002,13.62C-1.663,11.142 -3.729,6.267 -4.224,5.276C-4.72,4.285 -4.637,3.541 -6.703,2.715C-8.768,1.889 -15.046,0.898 -15.542,0.815C-15.817,0.769 -18.911,0.343 -21.811,0C-20.973,3.571 -20.103,7.396 -19.901,8.726C-19.488,11.452 -17.919,17.566 -17.919,17.566C-17.919,17.566 -5.466,18.179 -2.91,18.341C-2.071,18.394 -1.064,18.434 0,18.465",
    transform: "translate(105.678 46.543)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-6.015L-15.784,-6.725C-15.784,-6.725 -14.297,-1.603 -14.049,-1.438C-13.801,-1.273 -6.366,0.214 -1.327,0.462C3.713,0.71 9.413,-0.034 10.404,-0.116C11.396,-0.199 11.726,-0.116 11.809,-0.529C11.891,-0.942 11.231,-5.734 11.231,-5.734L0,-6.015Z",
    transform: "translate(103.543 70.835)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,-6.115C-0.126,-5.548 -0.315,-4.728 -0.378,-4.287C-0.441,-3.845 0,-1.891 0.441,-1.45C0.883,-1.008 2.143,-0.252 5.863,-0.126C9.583,0 10.654,-0.126 10.907,-0.441C11.159,-0.756 11.285,-1.576 11.285,-1.576L0,-6.115Z",
    transform: "translate(85.662 79.624)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,-4.917C-0.252,-4.287 -1.513,-2.584 -2.963,-1.954C-4.413,-1.324 -4.539,-1.45 -4.035,-1.134C-3.531,-0.819 -3.467,-0.252 -2.774,-0.126C-2.081,0 0.883,-0.063 0.883,-0.063C0.883,-0.063 1.072,-1.197 0.946,-2.269C0.82,-3.341 0.126,-4.917 0,-4.917",
    transform: "translate(115.167 79.12)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,8.322C0.757,7.881 3.026,4.539 3.846,2.585C4.665,0.631 4.665,0 4.665,0C4.665,0 7.628,1.892 10.592,1.702C13.555,1.513 17.274,0.252 17.274,0.252L17.968,-0.252C17.968,-0.252 18.976,1.072 19.481,2.585C19.985,4.098 19.796,6.116 19.733,6.683L19.67,7.25C19.67,7.25 4.602,8.574 0,8.322",
    transform: "translate(92.723 78.11)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,-20.88C-1.304,-21.206 -8.148,-20.446 -10.429,-20.12C-12.696,-19.796 -56.277,-16.897 -57.23,-16.862C-57.232,-15.882 -57.313,-7.892 -57.086,-7.178C-56.953,-6.757 -56.545,-5.631 -56.545,-5.631C-56.545,-5.631 -51.865,-4.393 -51.401,-4.354C-50.937,-4.316 -47.185,-4.122 -47.185,-4.122C-47.185,-4.122 -47.34,-5.747 -47.301,-7.603C-47.263,-9.459 -46.683,-12.012 -45.406,-12.205C-44.13,-12.399 -43.589,-10.929 -43.357,-10.194C-43.125,-9.459 -42.622,-7.99 -42.39,-6.481C-42.158,-4.973 -42.158,-4.316 -42.119,-4.161C-42.08,-4.006 -41.81,-3.658 -41.81,-3.658L-4.683,-1.608C-4.683,-1.608 -4.915,-5.553 -4.837,-6.674C-4.76,-7.796 -4.334,-10.581 -2.788,-11.973C-1.241,-13.365 1.272,-13.309 2.278,-12.11C3.276,-11.27 4.158,-9.622 4.367,-8.028C4.48,-7.166 4.986,-3.967 5.025,-3.58C5.063,-3.194 5.334,-1.84 5.373,-1.685C5.411,-1.531 5.837,-0.835 5.837,-0.835L24.942,0.171C24.942,0.171 27.108,0.287 27.262,0.326C26.761,-0.995 26.761,-12.728 26.761,-12.728C26.761,-12.728 1.304,-20.554 0,-20.88",
    transform: "translate(57.573 85.734)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,39.827C-0.007,36.891 -0.01,33.689 -0.066,31.569C-0.284,23.313 -1.886,11.96 -4.765,4.503C-5.246,3.258 -6.248,2.294 -7.506,1.852C-10.101,0.941 -15.124,0 -22.975,0.882C-25.955,1.218 -30.582,1.963 -30.582,1.963C-36.188,3.343 -73.357,12.37 -80.338,14.277C-80.79,14.4 -81.133,14.765 -81.222,15.224C-81.643,17.405 -81.806,22.677 -81.806,23.869C-71.66,22.719 -45.245,18.748 -35.827,17.433C-29.845,16.598 -27.373,19.601 -25.826,20.99C-22.913,23.605 -20.185,26.976 -20.06,29.846C-20.054,30.001 -20.046,30.179 -20.037,30.372C-12.093,32.789 -0.021,36.5 -0.021,36.5C-0.021,36.5 -0.021,37.92 0,39.827",
    transform: "translate(84.355 36.506)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,14.018C1.373,14.143 16.977,15.58 17.227,14.831C17.136,14.048 17.125,11.133 17.117,7.894C17.096,5.987 17.096,4.567 17.096,4.567C17.096,4.567 5.024,0.856 -2.919,-1.562C-2.769,1.805 -2.351,10.503 -2.351,11.329C-2.351,12.994 -1.373,13.893 0,14.018",
    transform: "translate(67.238 68.439)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E9E9E9",
    fillRule: "nonzero",
    d: "M0,9.64C-0.655,10.108 -2.153,12.541 -2.153,12.541C-2.153,12.541 6.364,10.576 7.113,10.389C7.862,10.202 20.778,5.99 28.92,5.148C37.063,4.306 41.275,4.493 42.959,4.961C44.644,5.429 48.669,7.113 48.669,7.113L65.235,9.173C65.235,9.173 69.353,5.803 60.929,3.089C52.506,0.375 40.807,-2.901 30.979,-0.561C21.152,1.778 0,9.64 0,9.64",
    transform: "translate(36.446 28.578)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-1.325,-1.325C-3.01,-1.325 -4.374,-0.731 -4.374,0.001C-4.374,0.732 -3.01,1.325 -1.325,1.325C0.359,1.325 1.724,0.732 1.724,0.001C1.724,-0.731 0.359,-1.325 -1.325,-1.325",
    transform: "matrix(0 -1 -1 0 12.39 78.705)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.197,-2.197C-4.331,-2.197 -6.061,-1.213 -6.061,0C-6.061,1.213 -4.331,2.197 -2.197,2.197C-0.063,2.197 1.667,1.213 1.667,0C1.667,-1.213 -0.063,-2.197 -2.197,-2.197",
    transform: "matrix(0 -1 -1 0 57.844 79.671)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,-10.484C1.023,-10.931 2.237,-11.507 2.621,-11.571C3.005,-11.635 5.562,-12.018 6.52,-11.89C7.479,-11.762 8.694,-11.379 9.205,-10.036C9.717,-8.694 10.1,-5.498 10.228,-4.667C10.356,-3.836 10.803,-1.087 10.612,-0.511C10.42,0.064 10.292,1.023 9.781,1.278C9.269,1.534 8.374,1.47 7.415,0.895C6.456,0.32 6.393,0.447 6.073,-0.575C5.753,-1.598 5.242,-4.922 5.37,-5.498C5.498,-6.073 5.817,-6.457 5.881,-7.032C5.945,-7.607 5.434,-9.142 5.434,-9.142C5.434,-9.142 3.132,-9.014 2.429,-8.694C1.726,-8.374 0.192,-7.607 0.192,-7.607C0.192,-7.607 -0.511,-10.037 0,-10.484",
    transform: "translate(79.589 46.661)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#767777",
    fillRule: "nonzero",
    d: "M0,-10.42C0.32,-10.484 5.817,-11.187 7.032,-11.251C8.246,-11.315 11.187,-10.74 12.529,-7.863C13.872,-4.986 14.447,-2.365 14.255,-1.342C14.064,-0.32 13.936,-0.064 13.488,0.192C13.041,0.448 11.954,0.895 11.507,0.575C11.059,0.256 11.187,0.575 10.484,-0.511C9.781,-1.598 9.972,-4.475 9.333,-5.945C8.694,-7.415 8.758,-8.31 7.927,-8.63C7.096,-8.95 2.621,-8.119 2.621,-8.119L0,-10.42Z",
    transform: "translate(105.415 50.752)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E9E9E9",
    fillRule: "nonzero",
    d: "M-13.39,14.533L15.471,14.533L15.505,13.209L-14.743,13.07L-13.39,14.533Z",
    transform: "scale(-1 1) rotate(-88.606 -11.964 65.371)"
  }), /*#__PURE__*/React.createElement("defs", null));
}
export default TransportPublicBus;
