// @bookingcom/bui-assets-react/transport/TransportTaxiTypeLarge.js
import * as React from "react";
function TransportTaxiTypeLarge() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-54.642,-53.849C-59.912,-53.849 -64.184,-29.383 -64.183,0.799C-64.183,30.98 -59.912,55.447 -54.642,55.447C-49.372,55.447 -45.1,30.98 -45.101,0.798C-45.101,-29.382 -49.372,-53.849 -54.642,-53.849",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(89.162 46.66 16.816)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M0,-18.328C-3.711,-17.025 -7.36,-13.282 -7.237,-10.028C-7.113,-6.774 -6.006,-0.844 -6.006,-0.844L3.649,0C3.649,0 4.639,-16.661 3.711,-17.424C2.783,-18.187 0,-18.328 0,-18.328",
    transform: "translate(63.255 85.06)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-19.557C-1.229,-19.333 -3.576,-18.551 -4.638,-15.199C-5.456,-12.614 -5.923,-9.387 -5.029,-6.09C-4.135,-2.794 -2.794,-1.006 -1.229,-0.503C0.335,0 4.358,-0.447 4.805,-0.726C5.253,-1.006 7.544,-2.961 7.767,-6.537C7.991,-10.114 7.376,-16.204 5.867,-17.769C4.358,-19.333 0,-19.557 0,-19.557",
    transform: "translate(17.177 87.14)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-20.004C-1.565,-19.278 -6.147,-16.093 -5.141,-7.208C-4.135,1.677 1.453,1.788 1.9,1.844C2.347,1.9 6.985,1.788 7.376,1.621C7.767,1.453 9.611,0.391 10.505,-1.006C11.399,-2.402 12.293,-3.464 12.461,-6.984C12.628,-10.505 12.852,-16.987 9.332,-19.445C5.811,-21.904 1.397,-20.786 0,-20.004",
    transform: "translate(63.556 89.263)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-4.463C0.338,-3.246 1.217,-1.488 2.299,-0.474C3.38,0.54 3.989,1.622 6.761,1.825C9.533,2.028 12.035,1.284 12.643,0.946C13.252,0.608 14.536,-0.001 15.483,-1.961C16.429,-3.922 16.564,-5.274 16.497,-5.883C16.429,-6.491 0,-4.463 0,-4.463",
    transform: "translate(93.89 87.337)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-42.011C-0.725,-41.879 -1.369,-41.462 -1.752,-40.832C-3.846,-37.387 -7.647,-26.794 -8.236,-23.841C-8.537,-22.331 -7.93,-15.185 -8.083,-14.879C-8.236,-14.573 -8.61,-13.548 -8.61,-11.82C-8.61,-11.351 -8.251,-6.771 -8.079,-4.594C-8.024,-3.909 -7.552,-3.339 -6.889,-3.159C-5.295,-2.728 -2.22,-1.878 -1.839,-1.769C-1.628,-2.343 -2.714,-6.207 -0.118,-10.887C2.593,-15.772 6.163,-11.662 7.312,-8.215C8.461,-4.769 8.587,0.267 8.74,0.497C19.88,1.779 45.167,3.451 45.167,3.451C45.167,3.451 43.823,-9.735 50.11,-11.65C52.79,-12.033 55.923,-11.916 57.592,-6.296C58.896,-1.903 59.856,3.733 59.856,3.733C59.856,3.733 61.388,3.504 62.001,3.504C62.613,3.504 90.111,2.661 90.723,2.661C91.336,2.661 101.217,1.895 101.6,1.819C101.983,1.742 102.596,1.359 102.519,1.053C102.442,0.746 102.059,-0.173 102.136,-0.632C102.213,-1.092 102.65,-5.475 102.65,-5.858C102.593,-7.103 100.221,-15.568 100.068,-16.028C99.915,-16.488 98.331,-20.25 98.153,-20.7C97.442,-22.499 87.966,-28.896 87.124,-29.279C86.281,-29.662 73.26,-39.466 72.188,-40.232C71.115,-40.998 67.907,-42.994 65.557,-43.478C60.585,-44.501 53.239,-44.573 45.512,-44.998C31.957,-45.744 11.282,-44.069 0,-42.011",
    transform: "translate(13.61 81.178)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-13.046C0,-13.046 6.469,-6.503 7.952,-4.988C9.718,-3.183 11.643,-0.848 14.082,-0.318C16.52,0.212 39.989,-0.323 39.565,-0.641C39.141,-0.959 36.154,-3.678 32.549,-6.328C28.944,-8.979 25.127,-11.412 24.155,-11.668C22.492,-12.105 2.757,-13.258 0,-13.046",
    transform: "translate(56.104 52.983)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#474646",
    fillRule: "nonzero",
    d: "M0,7.92C-0.686,7.379 -1.407,6.822 -2.166,6.264C-5.771,3.613 -9.588,1.181 -10.561,0.925C-11.563,0.661 -19.124,0.138 -25.58,-0.19C-24.184,1.225 -22.871,2.556 -22.3,3.14C-20.533,4.945 -18.608,7.28 -16.17,7.81C-14.792,8.11 -6.702,8.069 0,7.92",
    transform: "translate(86.356 44.855)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-12.164C-0.376,-10.578 -0.934,-8.222 -1.414,-0.125C-1.414,-0.125 10.48,0.184 14.158,0.305C17.775,0.305 17.244,-1.034 16.493,-2.22C15.967,-3.049 14.617,-5.53 11.083,-8.377C7.55,-11.223 5.848,-11.88 4.572,-12.175C3.296,-12.469 0,-12.164 0,-12.164",
    transform: "translate(42.558 52.72)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-9.395C-2.237,-9.09 -2.211,-7.87 -2.923,-6.345C-3.635,-4.82 -5.77,1.382 -5.77,1.382L4.909,1.567C4.909,1.567 12.659,1.567 14.679,1.526C16.686,1.485 18.242,1.035 19.02,-1.954C19.326,-3.131 19.816,-5.747 20.248,-8.178C20.576,-10.021 19.716,-10.962 17.668,-10.88C11.631,-10.639 1.904,-9.655 0,-9.395",
    transform: "translate(17.294 51.407)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-9.284C0.253,-7.934 0.928,-3.798 1.097,-3.46C1.266,-3.123 7.849,-1.857 8.44,-1.857C9.031,-1.857 19.75,0.422 19.75,0.085C19.75,-0.253 18.062,-4.642 17.64,-5.317C17.218,-5.992 16.29,-7.934 15.614,-8.187C14.939,-8.44 0.422,-9.706 0,-9.284",
    transform: "translate(70.504 67.254)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-8.018C0.169,-7.68 0.844,-4.98 0.844,-4.389C0.844,-3.798 1.182,0.338 1.182,0.338C1.182,0.338 4.136,-1.013 4.136,-1.519C4.136,-2.025 3.292,-4.726 2.87,-5.655C2.448,-6.583 1.857,-8.356 0,-8.018",
    transform: "translate(109.497 67.085)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-9.588C0.253,-9.166 3.798,-0.726 3.798,-0.726C3.798,-0.726 12.238,0.456 13.757,0.287C15.277,0.118 19.666,-0.726 19.666,-0.726C19.666,-0.726 19.666,-7.816 18.568,-9.335C18.568,-9.335 16.151,-9.838 11.901,-9.926C6.14,-10.044 0.097,-9.588 0,-9.588",
    transform: "translate(89.24 68.487)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#686868",
    fillRule: "nonzero",
    d: "M0,-10.279C-0.81,-10.225 -2.773,-10.109 -7.564,-9.846C-12.317,-9.585 -22.981,-9.846 -24.895,-9.959C-26.808,-10.071 -28.946,-10.184 -30.296,-9.284C-31.403,-8.546 -32.475,-6.793 -33.822,-4.737C-35.913,-1.634 -38.88,-0.474 -40.308,-0.408C-40.308,-0.408 -36.705,0.057 -36.373,0.057C-34.91,0.057 0.566,-0.487 1.698,-2.298C1.359,-2.977 0.965,-5.646 1.415,-6.996C1.755,-9.656 0.849,-10.336 0,-10.279",
    transform: "translate(113.77 85.316)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.571,-2.571C-5.595,-2.571 -8.047,-1.42 -8.047,-0.001C-8.047,1.419 -5.595,2.571 -2.571,2.571C0.454,2.571 2.906,1.419 2.906,-0.001C2.906,-1.42 0.454,-2.571 -2.571,-2.571",
    transform: "matrix(0 -1 -1 0 16.004 75.63)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-3.073,-3.073C-6.114,-3.073 -8.577,-1.697 -8.577,-0C-8.577,1.698 -6.114,3.073 -3.073,3.073C-0.033,3.073 2.431,1.698 2.431,-0C2.431,-1.697 -0.033,-3.073 -3.073,-3.073",
    transform: "matrix(0 -1 -1 0 64.45 77.725)"
  }));
}
export default TransportTaxiTypeLarge;
