// @bookingcom/bui-assets-react/transport/TransportTaxiTypeLargeExec.js
import * as React from "react";
function TransportTaxiTypeLargeExec() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-54.521,-53.73C-59.091,-53.73 -62.796,-29.317 -62.796,0.797C-62.796,30.912 -59.091,55.325 -54.521,55.325C-49.952,55.325 -46.247,30.912 -46.246,0.798C-46.246,-29.317 -49.951,-53.73 -54.521,-53.73",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(89.161 47.187 16.431)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M0,-18.289C-3.703,-16.989 -7.344,-13.254 -7.221,-10.007C-7.097,-6.761 -5.993,-0.843 -5.993,-0.843L3.641,0C3.641,0 4.629,-16.625 3.703,-17.387C2.777,-18.148 0,-18.289 0,-18.289",
    transform: "translate(62.169 84.685)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-19.515C-1.227,-19.292 -3.568,-18.511 -4.628,-15.166C-5.444,-12.588 -5.91,-9.367 -5.018,-6.078C-4.126,-2.788 -2.788,-1.004 -1.227,-0.502C0.335,0 4.349,-0.446 4.795,-0.725C5.241,-1.004 7.527,-2.955 7.75,-6.524C7.973,-10.092 7.36,-16.17 5.854,-17.731C4.349,-19.292 0,-19.515 0,-19.515",
    transform: "translate(16.192 86.76)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-19.96C-1.561,-19.235 -6.133,-16.057 -5.13,-7.192C-4.126,1.673 1.45,1.785 1.896,1.841C2.342,1.896 6.97,1.785 7.36,1.618C7.75,1.45 9.59,0.391 10.482,-1.003C11.374,-2.397 12.266,-3.456 12.434,-6.969C12.601,-10.481 12.824,-16.949 9.311,-19.402C5.799,-21.856 1.394,-20.741 0,-19.96",
    transform: "translate(62.47 88.878)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-4.452C0.337,-3.238 1.214,-1.484 2.294,-0.472C3.373,0.54 3.98,1.62 6.746,1.822C9.512,2.024 12.008,1.282 12.616,0.945C13.223,0.608 14.505,0.001 15.449,-1.956C16.393,-3.912 16.528,-5.262 16.461,-5.869C16.393,-6.476 0,-4.452 0,-4.452",
    transform: "translate(92.737 86.955)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-41.92C-0.723,-41.788 -1.366,-41.372 -1.748,-40.744C-3.837,-37.306 -7.631,-26.736 -8.218,-23.789C-8.519,-22.283 -7.913,-15.153 -8.066,-14.847C-8.218,-14.541 -8.591,-13.519 -8.591,-11.795C-8.591,-11.327 -8.233,-6.756 -8.061,-4.585C-8.007,-3.901 -7.536,-3.332 -6.874,-3.153C-5.284,-2.722 -2.215,-1.874 -1.835,-1.766C-1.624,-2.338 -2.708,-6.194 -0.118,-10.863C2.587,-15.738 6.15,-11.637 7.296,-8.198C8.443,-4.759 8.568,0.266 8.721,0.495C19.837,1.775 45.069,3.443 45.069,3.443C45.069,3.443 43.728,-9.714 50.001,-11.625C52.676,-12.008 55.802,-11.891 57.466,-6.283C58.768,-1.899 59.726,3.725 59.726,3.725C59.726,3.725 61.254,3.496 61.866,3.496C62.477,3.496 89.915,2.655 90.526,2.655C91.137,2.655 100.997,1.891 101.379,1.814C101.761,1.738 102.372,1.356 102.296,1.05C102.219,0.744 101.837,-0.173 101.914,-0.632C101.99,-1.09 102.426,-5.463 102.426,-5.846C102.37,-7.088 100.003,-15.535 99.85,-15.994C99.697,-16.452 98.117,-20.206 97.939,-20.656C97.23,-22.45 87.775,-28.833 86.934,-29.215C86.093,-29.598 73.1,-39.38 72.031,-40.145C70.961,-40.909 67.759,-42.901 65.414,-43.383C60.453,-44.405 53.123,-44.477 45.413,-44.901C31.887,-45.645 11.258,-43.974 0,-41.92",
    transform: "translate(12.632 80.81)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#BDBDBD",
    fillRule: "nonzero",
    d: "M0,-13.017C0,-13.017 6.454,-6.489 7.935,-4.977C9.697,-3.176 11.618,-0.846 14.051,-0.317C16.484,0.212 39.902,-0.322 39.479,-0.639C39.056,-0.956 36.075,-3.669 32.478,-6.314C28.881,-8.959 25.073,-11.386 24.102,-11.642C22.443,-12.078 2.751,-13.229 0,-13.017",
    transform: "translate(55.034 52.676)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,8.777C-0.978,7.982 -2.077,7.116 -3.256,6.249C-6.853,3.604 -10.662,1.176 -11.633,0.921C-12.656,0.652 -20.539,0.112 -27.076,-0.214C-25.376,1.508 -23.593,3.315 -22.9,4.023C-21.138,5.824 -19.217,8.154 -16.784,8.683C-15.364,8.991 -6.804,8.938 0,8.777",
    transform: "translate(85.869 43.677)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-12.137C-0.375,-10.555 -0.932,-8.203 -1.411,-0.124C-1.411,-0.124 10.458,0.184 14.127,0.305C17.736,0.305 17.206,-1.032 16.457,-2.215C15.933,-3.042 14.585,-5.518 11.059,-8.358C7.533,-11.198 5.835,-11.854 4.562,-12.148C3.289,-12.442 0,-12.137 0,-12.137",
    transform: "translate(41.517 52.414)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-9.375C-2.232,-9.071 -2.207,-7.853 -2.917,-6.331C-3.627,-4.81 -5.757,1.379 -5.757,1.379L4.898,1.563C4.898,1.563 12.631,1.564 14.647,1.522C16.649,1.482 18.202,1.032 18.978,-1.95C19.284,-3.124 19.773,-5.735 20.204,-8.161C20.531,-9.999 19.673,-10.939 17.63,-10.857C11.606,-10.616 1.9,-9.634 0,-9.375",
    transform: "translate(16.308 51.105)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-9.264C0.253,-7.917 0.926,-3.79 1.095,-3.453C1.263,-3.116 7.832,-1.853 8.422,-1.853C9.011,-1.853 19.707,0.421 19.707,0.084C19.707,-0.253 18.023,-4.632 17.602,-5.306C17.18,-5.979 16.254,-7.917 15.58,-8.169C14.907,-8.422 0.421,-9.685 0,-9.264",
    transform: "translate(69.402 66.917)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-8.001C0.168,-7.664 0.842,-4.969 0.842,-4.38C0.842,-3.79 1.179,0.337 1.179,0.337C1.179,0.337 4.127,-1.011 4.127,-1.516C4.127,-2.022 3.285,-4.716 2.863,-5.643C2.442,-6.569 1.853,-8.338 0,-8.001",
    transform: "translate(108.31 66.75)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-9.567C0.253,-9.146 3.79,-0.724 3.79,-0.724C3.79,-0.724 12.212,0.455 13.728,0.287C15.243,0.118 19.623,-0.724 19.623,-0.724C19.623,-0.724 19.623,-7.798 18.528,-9.314C18.528,-9.314 16.116,-9.817 11.875,-9.904C6.127,-10.022 0.097,-9.567 0,-9.567",
    transform: "translate(88.098 68.147)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#383838",
    fillRule: "nonzero",
    d: "M0,-10.257C-0.808,-10.203 -2.767,-10.088 -7.547,-9.825C-12.29,-9.565 -22.931,-9.825 -24.84,-9.937C-26.749,-10.05 -28.883,-10.162 -30.23,-9.264C-31.334,-8.528 -32.404,-6.778 -33.748,-4.727C-35.835,-1.63 -38.796,-0.474 -40.22,-0.407C-40.22,-0.407 -36.625,0.056 -36.294,0.056C-34.834,0.056 0.565,-0.486 1.694,-2.293C1.356,-2.971 0.963,-5.634 1.412,-6.981C1.751,-9.636 0.847,-10.313 0,-10.257",
    transform: "translate(112.573 84.94)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.565,-2.565C-5.583,-2.565 -8.029,-1.417 -8.029,0C-8.029,1.416 -5.583,2.565 -2.565,2.565C0.453,2.565 2.899,1.416 2.899,0C2.899,-1.417 0.453,-2.565 -2.565,-2.565",
    transform: "matrix(0 -1 -1 0 15.02 75.274)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-3.067,-3.067C-6.1,-3.067 -8.559,-1.694 -8.559,0.001C-8.559,1.693 -6.1,3.067 -3.067,3.067C-0.034,3.067 2.426,1.693 2.426,0.001C2.426,-1.694 -0.034,-3.067 -3.067,-3.067",
    transform: "matrix(0 -1 -1 0 63.361 77.365)"
  }));
}
export default TransportTaxiTypeLargeExec;
