// @bookingcom/bui-assets-react/transport/TransportTaxiTypeMinibus.js
import * as React from "react";
function TransportTaxiTypeMinibus() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-56.702,-55.918C-59.041,-55.889 -60.936,-30.476 -60.936,0.842C-60.937,32.161 -59.041,57.526 -56.702,57.496C-54.363,57.468 -52.468,32.055 -52.468,0.736C-52.468,-30.583 -54.363,-55.948 -56.702,-55.918",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(89.203 45.528 15.247)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-6.083C0.174,-5.04 0.695,-2.868 2.52,-1.564C4.345,-0.261 5.214,0.087 6.691,0.087C8.168,0.087 11.47,-0.261 11.992,-0.695C12.513,-1.13 13.469,-1.043 14.512,-3.042C15.554,-5.04 15.728,-6.17 15.728,-6.17L0,-6.083Z",
    transform: "translate(97.21 88.966)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-17.589C-1.588,-16.596 -3.434,-14.239 -3.769,-11.81C-4.104,-9.381 -3.769,-5.025 -3.015,-3.099C-2.262,-1.173 -0.754,0.586 0.419,0.837C1.591,1.089 8.054,0.838 8.891,0.503C9.729,0.167 10.65,-0.168 11.32,-1.424C11.99,-2.68 12.828,-3.769 13.163,-6.868C13.498,-9.967 12.577,-17.003 10.818,-17.84C9.059,-18.678 3.015,-18.092 2.094,-18.092C1.173,-18.092 0.67,-18.008 0,-17.589",
    transform: "translate(17.288 85.242)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.262,-2.262C-4.945,-2.262 -7.119,-1.249 -7.119,-0C-7.119,1.249 -4.945,2.262 -2.262,2.262C0.422,2.262 2.596,1.249 2.596,-0C2.596,-1.249 0.422,-2.262 -2.262,-2.262",
    transform: "matrix(0 -1 -1 0 18.042 74.437)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-18.982C-1.588,-17.911 -3.434,-15.366 -3.769,-12.745C-4.104,-10.124 -3.769,-5.424 -3.015,-3.345C-2.262,-1.266 -0.754,0.633 0.419,0.904C1.591,1.175 8.054,0.904 8.891,0.542C9.729,0.181 10.65,-0.181 11.32,-1.537C11.99,-2.893 12.828,-4.068 13.163,-7.412C13.498,-10.757 12.577,-18.349 10.818,-19.253C9.059,-20.157 3.015,-19.524 2.094,-19.524C1.173,-19.524 0.67,-19.434 0,-18.982",
    transform: "translate(71.346 88.332)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.262,-2.262C-5.156,-2.262 -7.505,-1.249 -7.505,-0C-7.505,1.249 -5.156,2.262 -2.262,2.262C0.633,2.262 2.982,1.249 2.982,-0C2.982,-1.249 0.633,-2.262 -2.262,-2.262",
    transform: "matrix(0 -1 -1 0 72.1 76.85)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#D9D9D9",
    fillRule: "nonzero",
    d: "M0,-43.615C-1.571,-43.296 -2.868,-42.244 -3.539,-40.788C-7.38,-32.448 -8.22,-12.612 -8.542,-6.132C-8.61,-4.749 -7.857,-3.476 -6.627,-2.84C-2.92,-0.921 4.411,1.549 4.411,1.549C3.407,-10.066 11.415,-15.106 13.625,-3.69C13.887,-2.337 14.498,2.728 15.502,2.379C15.502,2.379 52.243,5.342 58.453,5.446C57.16,-2.839 60.129,-8.542 64.099,-8.73C67.399,-8.886 71.564,-5.161 73.283,2.215C74.281,3.825 74.409,6.178 76.112,6.647C76.198,6.647 82.28,6.966 88.365,6.767C96.232,6.509 104.589,6.157 105.063,5.686C108.328,2.594 109.752,0.412 109.634,-4.425L110.094,-4.688C110.075,-5.024 109.545,-5.85 109.371,-6.134L108.451,-6.331C109.485,-17.585 105.617,-18.191 96.618,-23.027C93.502,-26.398 80.027,-47.184 72.298,-48.401C66.477,-49.316 40.054,-50.581 0,-43.615",
    transform: "translate(9.226 77.645)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-8.702C0,-8.702 -0.657,-7.635 -0.903,-5.665C-1.149,-3.694 -2.052,1.806 -2.134,3.119C-2.217,4.433 -2.217,5.172 -2.217,5.172C-2.217,5.172 39.979,4.597 40.39,4.433C40.8,4.269 41.785,3.448 41.867,2.873C41.949,2.298 43.92,-10.098 44.084,-10.836C44.248,-11.575 44.494,-13.135 44.33,-13.463C44.166,-13.792 44.002,-13.874 42.853,-13.874C41.703,-13.874 9.851,-11.001 3.284,-9.933C1.258,-9.604 0.41,-9.687 0,-8.702",
    transform: "translate(8.446 49.146)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-17.528C0,-17.528 4.407,-8.576 5.31,-6.935C6.213,-5.293 9.182,0.329 9.182,0.329L34.207,0.647C34.207,0.647 24.875,-11.653 20.92,-15.947C20.315,-16.591 19.507,-16.995 18.627,-17.08C10.484,-17.866 4.378,-18.175 0,-17.528",
    transform: "translate(67.626 53.565)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#474646",
    fillRule: "nonzero",
    d: "M0,-10.319C-0.604,-10.963 -1.413,-11.367 -2.293,-11.452C-6.801,-11.887 -10.664,-12.167 -13.976,-12.229C-12.673,-9.603 -11.108,-6.472 -10.61,-5.566C-9.707,-3.924 -6.738,1.698 -6.738,1.698L9.934,1.91C6.857,-2.062 2.423,-7.689 0,-10.319",
    transform: "translate(83.546 52.196)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-4.105L0.657,-1.396C1.067,-0.821 1.478,0.328 3.776,0.328C6.075,0.328 10.59,-0.082 11.575,-0.165C12.56,-0.247 13.299,-0 13.627,-0.739C13.956,-1.478 14.448,-4.433 14.448,-4.433L0,-4.105Z",
    transform: "translate(100.772 77.416)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M2.16,-11.736L11.405,-11.736L14.741,2.839C14.741,2.839 9.429,4.09 6.402,4.307L2.16,-11.736Z",
    transform: "scale(1 -1) rotate(-75.965 17.915 -97.245)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M5.815,-14.189C6.181,-13.817 9.324,-7.166 9.428,-4.587C9.555,-1.402 5.815,2.384 5.815,2.384C5.815,2.384 3.429,-0.897 1.93,-3.292C1.08,-4.649 5.815,-14.189 5.815,-14.189",
    transform: "scale(-1 1) rotate(44.575 -126.96 -88.019)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,15.215C-0.765,13.418 -2.801,8.447 -3.947,6.161C-3.947,6.161 -6.265,1.782 -6.64,0.95C-7.017,0.117 -7.518,-0.529 -8.508,-1.197C-9.503,-1.868 -11.936,-1.807 -12.876,-1.893C-13.413,-1.942 -14.132,-1.909 -14.654,-1.871C-15.193,-1.831 -15.651,-1.416 -15.768,-0.847C-15.825,-0.567 -15.853,-0.159 -15.829,0.504C-15.764,2.307 -15.801,6.787 -15.654,8.425C-15.506,10.064 -15.934,13.203 -14.017,14.07C-12.12,14.927 -4.044,16.544 -4.044,16.544L-4.044,16.522C-3.141,16.723 -1.963,17.001 -1.674,17.046C-0.948,17.157 0.763,17.006 0,15.215",
    transform: "translate(71.128 37.313)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M-0.451,-3.781C0.533,-5.454 1.968,-6.965 1.911,-6.246C1.855,-5.527 -0.451,3.368 -0.451,3.368C-0.451,3.368 -1.367,3.257 -1.89,2.252C-2.414,1.247 -2.214,-0.787 -0.451,-3.781",
    transform: "scale(-1 1) rotate(-4.501 783.801 1509.863)"
  }));
}
export default TransportTaxiTypeMinibus;
