// @bookingcom/bui-assets-react/transport/TransportTaxiTypeStandardElectric.js
import * as React from "react";
function TransportTaxiTypeStandardElectric() {
  return /*#__PURE__*/React.createElement("svg", {
    xmlns: "http://www.w3.org/2000/svg",
    xmlSpace: "preserve",
    fillRule: "evenodd",
    strokeLinejoin: "round",
    strokeMiterlimit: 2,
    clipRule: "evenodd",
    viewBox: "0 0 120 120"
  }, /*#__PURE__*/React.createElement("path", {
    fill: "#636566",
    fillRule: "nonzero",
    d: "M-50.838,-49.349C-55.859,-49.349 -59.929,-26.578 -59.929,1.511C-59.929,29.601 -55.859,52.371 -50.838,52.371C-45.816,52.371 -41.746,29.6 -41.747,1.512C-41.746,-26.578 -45.816,-49.349 -50.838,-49.349",
    opacity: 0.47,
    transform: "scale(1 -1) rotate(88.298 48.34 15.414)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#7ED84E",
    fillRule: "nonzero",
    d: "M-18.216,-18.216C-8.155,-18.216 0.001,-10.06 0.001,0.001C0.001,10.061 -8.155,18.216 -18.216,18.216C-28.276,18.216 -36.432,10.061 -36.432,0.001C-36.432,-10.06 -28.276,-18.216 -18.216,-18.216",
    transform: "matrix(0 -1 -1 0 94.033 26.654)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M0,34.369C0,35.156 8.048,15.673 8.048,15.673L-5.116,15.377C-5.116,15.377 3.642,0.524 3.642,-0.787L10.464,-0.787L2.175,10.996L14.776,12.001L0,34.369Z",
    transform: "translate(88.932 31.935)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-2.714,-15.686C-1.039,-14.009 1.697,-11.659 1.376,-8.94C1.057,-6.221 -1.146,-0.162 -2.714,0.429L-9.674,-2.13L-2.714,-15.686Z",
    transform: "scale(-1 1) rotate(-21.641 199.985 59.43)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#939393",
    fillRule: "nonzero",
    d: "M-5.231,-20.093C-3.622,-18.353 -0.356,-11.638 -0.781,-8.933C-1.206,-6.229 -3.64,-0.261 -5.231,0.271L-11.619,-2.156L-5.231,-20.093Z",
    transform: "scale(-1 1) rotate(-23.854 167.96 159.003)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,10.088C0.295,10.737 2.065,13.628 4.189,14.277C6.312,14.925 9.97,14.571 11.386,13.687C12.801,12.802 14.04,12.094 14.984,9.675C15.928,7.256 17.285,1.77 14.04,-0.826C10.796,-3.421 2.832,-4.837 2.006,-3.893C1.18,-2.949 0,10.088 0,10.088",
    transform: "translate(92.834 72.173)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-18.377C-1.06,-17.769 -2.238,-17.258 -3.187,-15.427C-4.136,-13.596 -4.611,-10.24 -4.543,-8.714C-4.476,-7.188 -3.967,-4.171 -2.679,-2.136C-1.39,-0.102 -0.271,0.848 1.356,1.085C2.984,1.322 7.076,0.981 7.788,0.811C8.5,0.642 9.483,0.031 9.958,-0.545C10.432,-1.122 11.483,-2.274 12.06,-4.919C12.636,-7.564 12.775,-11.978 9.995,-16.895C8.549,-19.451 2.306,-19.699 0,-18.377",
    transform: "translate(59.478 87.416)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-2.054,-2.037C-4.574,-2.057 -6.618,-1.155 -6.618,-0.021C-6.618,1.113 -4.574,2.05 -2.054,2.071C0.467,2.091 2.511,1.188 2.511,0.054C2.511,-1.08 0.467,-2.016 -2.054,-2.037",
    transform: "scale(1 -1) rotate(89.526 68.644 -7.841)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-15.987C-0.923,-15.458 -1.947,-15.014 -2.773,-13.421C-3.599,-11.828 -4.012,-8.908 -3.953,-7.58C-3.894,-6.253 -3.451,-3.628 -2.33,-1.858C-1.209,-0.088 -0.236,0.738 1.18,0.944C2.596,1.15 5.516,0.944 6.135,0.797C6.755,0.649 7.61,0.118 8.023,-0.383C8.436,-0.885 9.35,-1.888 9.852,-4.188C10.353,-6.489 10.678,-10.324 8.259,-14.601C7.001,-16.824 2.006,-17.137 0,-15.987",
    transform: "translate(14.255 83.618)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#E0E0E0",
    fillRule: "nonzero",
    d: "M0,-2.953C0.871,-9.548 4.38,-8.987 7.491,-10.356C10.099,-11.503 13.118,-14.745 17.845,-16.658C27.552,-20.587 46.762,-19.263 59.161,-18.783C68.573,-18.418 81.105,-9.243 84.408,-6.037C84.433,-6.012 84.455,-6.001 84.491,-5.996C84.631,-5.975 85.182,-5.849 87.174,-5.113C88.858,-4.491 92.153,-3.368 92.574,-3.329C92.994,-3.291 93.967,-3.329 95.783,-2.47C102.536,0.728 102.032,7.585 101.826,12.563C101.903,12.984 102.006,13.925 101.979,14.285C101.889,15.498 94.548,17.223 84.33,17.571C82.475,17.634 67.949,17.57 62.765,17.503C61.601,17.487 60.617,16.68 60.345,15.548C59.601,12.453 59.051,7.648 58.333,5.437C56.184,-1.183 47.38,0.585 46.714,8.005C46.286,12.775 46.897,14.732 47.448,16.628C39.61,16.229 22.129,14.916 13.935,13.937C13.334,13.866 12.201,13.175 11.972,12.028C10.97,7.274 12.919,0.6 7.169,-0.184C1.337,0.483 1.283,8.061 2.345,12.529C2.233,12.529 1.962,12.213 1.576,11.979C0.613,11.398 -0.04,9.985 -0.363,8.907C-0.831,7.348 -0.97,5.518 -0.878,4.056C-0.86,3.76 -0.765,3.485 -0.667,3.205C-0.074,1.504 -0.037,-2.114 0,-2.953",
    transform: "translate(8.656 67.492)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,4.933C-1.921,4.047 -0.964,2.987 -0.375,2.513C1.329,1.141 4.995,-1.902 10.56,-2.82C14.544,-3.477 19.74,-3.321 23.347,-0.452C27.251,2.654 31.307,7.334 31.768,8.157C30.034,8.41 4.932,7.207 0,4.933",
    transform: "translate(21.93 53.429)"
  }), /*#__PURE__*/React.createElement("path", {
    fillRule: "nonzero",
    d: "M0,-12.422C0,-12.422 4.533,-9.558 8.379,-5.945C11.07,-3.417 13.89,-1.024 14.25,-0.808C14.609,-0.593 28.19,0 31.861,-0.075C37.627,-0.191 43.8,-0.557 46.006,-0.635C46.166,-0.64 39.63,-5.136 35.087,-7.902C30.212,-10.869 29.508,-10.571 25.627,-11.361C21.746,-12.152 16.605,-12.366 10.999,-12.366C8.913,-12.366 0,-12.422 0,-12.422",
    transform: "translate(44.205 62.172)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#474646",
    fillRule: "nonzero",
    d: "M0,-7.852C-3.881,-8.643 -9.022,-8.857 -14.628,-8.857C-14.801,-8.857 -15.034,-8.857 -15.292,-8.858C-13.843,-7.774 -12.153,-6.422 -10.574,-4.939C-7.883,-2.411 -5.063,-0.018 -4.703,0.198C-4.344,0.414 9.237,1.006 12.908,0.932C14.422,0.901 15.963,0.853 17.454,0.798C15.185,-0.745 12.009,-2.841 9.459,-4.393C4.585,-7.36 3.881,-7.061 0,-7.852",
    transform: "translate(63.158 61.165)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-6.667C1.033,-3.642 4.833,-0.911 5.326,-0.786C6.756,-0.424 11.012,0.053 13.24,-0.162C14.028,-0.239 16.334,-0.467 16.689,-0.822C14.843,-1.77 8.903,-4.66 2.947,-6.164C0.743,-6.72 0,-6.667 0,-6.667",
    transform: "translate(67.888 71.47)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#A5AAAD",
    fillRule: "nonzero",
    d: "M0,-5.171C0.561,-4.703 1.282,-4.445 1.999,-3.208C2.706,-1.989 2.708,-0.528 3.153,-0.243C3.597,0.042 5.16,-0.152 5.522,-0.273C6.29,-0.528 5.923,-1.229 5.923,-1.229C5.923,-1.229 5.235,-2.925 4.044,-3.747C3.289,-4.268 0.461,-5.213 0,-5.171",
    transform: "translate(101.567 70.843)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#fff",
    fillRule: "nonzero",
    d: "M-1.648,-1.648C-3.954,-1.648 -5.823,-0.91 -5.823,0C-5.823,0.91 -3.954,1.648 -1.648,1.648C0.658,1.648 2.527,0.91 2.527,0C2.527,-0.91 0.658,-1.648 -1.648,-1.648",
    transform: "matrix(0 -1 -1 0 14.291 74.247)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#DEDEDE",
    fillRule: "nonzero",
    d: "M0,-11.461C-0.422,-10.807 -0.829,-10.322 -2.066,-7.636C-4.121,-3.171 -4.164,-0.467 -4.164,-0.467L-0.708,0C-0.708,0 -0.23,-2.714 0.22,-4.214C0.671,-5.715 2.08,-9.817 3.43,-11.456L0,-11.461Z",
    transform: "translate(35.172 61.23)"
  }), /*#__PURE__*/React.createElement("path", {
    fill: "#4F4F4F",
    fillRule: "nonzero",
    d: "M0,-1.407C-0.364,-1.491 -0.802,-1.342 -0.818,-0.917C-0.836,-0.445 -0.838,0.324 -0.836,0.532C1.356,0.824 6.262,1.059 9.794,0.83C15.308,0.472 19.515,-0.687 19.515,-0.687C19.515,-0.687 19.447,-1.232 19.336,-1.94C19.317,-2.277 19.07,-2.466 18.667,-2.416C17.622,-2.286 14.249,-1.709 10.734,-1.293C5.997,-0.732 1.508,-1.318 0,-1.407",
    transform: "translate(88.098 79.2)"
  }));
}
export default TransportTaxiTypeStandardElectric;
